EVS has worked in partnership with and been used by many other organizations in the cancer research and biomedical community. Some noteworthy examples are described briefly in this section as identified in the list that follows. Single-institution efforts are described first, followed by thematic efforts that span multiple institutions.Some projects not covered in this section have put listings on the
<a href="http://bioportal.bioontology.org/ontologies/50373#projects_content" rel="nofollow">
  NCBO BioPortal detail page for NCI Thesaurus
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
.
<ul>
  <li>
    <a href="#UserProfilesBroaderCommunity-Institutional" rel="nofollow">
      Institutional
    </a>
    <ul>
      <li>
        <a href="#UserProfilesBroaderCommunity-AmericanAssociationforCancerResearch(AACR)" rel="nofollow">
          American Association for Cancer Research (AACR)
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-Genomics,Evidence,Neoplasia,Information,Exchange(GENIE)" rel="nofollow">
              Genomics, Evidence, Neoplasia, Information, Exchange (GENIE)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-AmericanCollegeofCardiology(ACC)" rel="nofollow">
          American College of Cardiology (ACC)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-AmericanSocietyofClinicalOncology(ASCO)" rel="nofollow">
          American Society of Clinical Oncology (ASCO)
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-CancerLearningIntelligenceNetworkforQuality(CancerLinQ)" rel="nofollow">
              Cancer Learning Intelligence Network for Quality (CancerLinQ)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-CatalogueofSomaticMutationsinCancer(COSMIC)" rel="nofollow">
          Catalogue of Somatic Mutations in Cancer (COSMIC)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-ClinicalGenomeResource(ClinGen)" rel="nofollow">
          Clinical Genome Resource (ClinGen)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-DukeUniversity" rel="nofollow">
          Duke University
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-EmoryUniversity" rel="nofollow">
          Emory University
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-EuropeanBioinformaticsInstitute(EBI)" rel="nofollow">
          European Bioinformatics Institute (EBI)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-GeneralElectric(GE)" rel="nofollow">
          General Electric (GE)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-GeorgetownUniversity" rel="nofollow">
          Georgetown University
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-JacksonLaboratory" rel="nofollow">
          Jackson Laboratory
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-MayoClinic" rel="nofollow">
          Mayo Clinic
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-PharmacogenomicsResearchNetwork(PGRN)OntologyNetworkResource(PHONT)" rel="nofollow">
              Pharmacogenomics Research Network (PGRN) Ontology Network Resource (PHONT)
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-StrategicHealthITAdvancedResearchProjects:Area4-SecondaryUseofEHRData(SHARPn)" rel="nofollow">
              Strategic Health IT Advanced Research Projects: Area 4 - Secondary Use of EHR Data (SHARPn)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-MDAnderson" rel="nofollow">
          MD Anderson
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-MemorialSloanKetteringCancerCenter" rel="nofollow">
          Memorial Sloan Kettering Cancer Center
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-OncoKB" rel="nofollow">
              OncoKB
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-cBioPortal" rel="nofollow">
              cBioPortal
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-OncoTree" rel="nofollow">
              OncoTree
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-OhioStateUniversityMedicalCenter" rel="nofollow">
          Ohio State University Medical Center
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-SeoulNationalUniversity,Korea" rel="nofollow">
          Seoul National University, Korea
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-StanfordUniversity" rel="nofollow">
          Stanford University
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-Prot%C3%A9g%C3%A9" rel="nofollow">
              Protégé
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-NationalCenterforBiomedicalOntology(NCBO)" rel="nofollow">
              National Center for Biomedical Ontology (NCBO)
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-TerminologyMetadata" rel="nofollow">
              Terminology Metadata
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-TissueMicroarrayDatabase(TMAD)" rel="nofollow">
              Tissue Microarray Database (TMAD)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-SwissInstituteofBioinformatics(SIB)" rel="nofollow">
          Swiss Institute of Bioinformatics (SIB)
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-neXtProt" rel="nofollow">
              neXtProt
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-Cellosaurus" rel="nofollow">
              Cellosaurus
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-UniversityofPittsburgh" rel="nofollow">
          University of Pittsburgh
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-OntologyDevelopmentInformationExtraction(ODIE)" rel="nofollow">
              Ontology Development Information Extraction (ODIE)
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-TextInformationExtractionSystem(TIES)" rel="nofollow">
              Text Information Extraction System (TIES)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-WashingtonUniversity" rel="nofollow">
          Washington University
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-YaleUniversity" rel="nofollow">
          Yale University
        </a>
      </li>
    </ul>
  </li>
  <li>
    <a href="#UserProfilesBroaderCommunity-ProjectandTopical" rel="nofollow">
      Project and Topical
    </a>
    <ul>
      <li>
        <a href="#UserProfilesBroaderCommunity-AnimalModelsandMappings" rel="nofollow">
          Animal Models and Mappings
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-Uberon" rel="nofollow">
              Uberon
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-OntologyAlignmentEvaluationInitiative(OAEI)" rel="nofollow">
              Ontology Alignment Evaluation Initiative (OAEI)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-CommonBiorepositoryModel(CBM)" rel="nofollow">
          Common Biorepository Model (CBM)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-eMERGENetwork" rel="nofollow">
          eMERGE Network
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-eTOX" rel="nofollow">
          eTOX
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-GlobalAlignmentofImmunizationsafetyAssessmentinpregnancy(GAIA)" rel="nofollow">
          Global Alignment of Immunization safety Assessment in pregnancy (GAIA)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-HumanStudiesDatabase(HSDB)" rel="nofollow">
          Human Studies Database (HSDB)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-InternationalHumanEpigenomeConsortium(IHEC)" rel="nofollow">
          International Human Epigenome Consortium (IHEC)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-ImagingStandards" rel="nofollow">
          Imaging Standards
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-LexEVSAdopterCommunity" rel="nofollow">
          LexEVS Adopter Community
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-Nanotechnology" rel="nofollow">
          Nanotechnology
        </a>
        <ul>
          <li>
            <a href="#UserProfilesBroaderCommunity-NanotechnologyWorkingGroup" rel="nofollow">
              Nanotechnology Working Group
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-caNanoLab" rel="nofollow">
              caNanoLab
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-ISA-TAB-Nano" rel="nofollow">
              ISA-TAB-Nano
            </a>
          </li>
          <li>
            <a href="#UserProfilesBroaderCommunity-NanoParticleOntology(NPO)" rel="nofollow">
              NanoParticle Ontology (NPO)
            </a>
          </li>
        </ul>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-OpenBiomedicalOntologies(OBO)" rel="nofollow">
          Open Biomedical Ontologies (OBO)
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-PhenX" rel="nofollow">
          PhenX
        </a>
      </li>
      <li>
        <a href="#UserProfilesBroaderCommunity-TranslationalResearchandPatientSafetyinEurope(TRANSFoRm)" rel="nofollow">
          Translational Research and Patient Safety in Europe (TRANSFoRm)
        </a>
      </li>
    </ul>
  </li>
</ul>
<h2 id="UserProfilesBroaderCommunity-Institutional">
  Institutional
</h2>
<h3 id="UserProfilesBroaderCommunity-AmericanAssociationforCancerResearch(AACR)">
  American Association for Cancer Research (AACR)
</h3>
AACR, founded in 1907, fosters research in cancer and related biomedical science, including advances in the understanding of cancer etiology, prevention, diagnosis, and treatment throughout the world.  A key recent initiative related to EVS is GENIE.
<h4 id="UserProfilesBroaderCommunity-Genomics,Evidence,Neoplasia,Information,Exchange(GENIE)">
  Genomics, Evidence, Neoplasia, Information, Exchange (GENIE)
</h4>
AACR created
<a href="http://www.aacr.org/Research/Research/Pages/aacr-project-genie.aspx" rel="nofollow">
  GENIE
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
as a multi-phase, multi-year, international cancer registry built by sharing clinical cancer sequencing data from eight international institutions that are global leaders in genomic sequencing for clinical utility. An initial set of nearly 19,000 genomic records was released in January 2017 and can be directly accessed through cBioPortal. The primary cancer diagnosis code is based on OncoTree, which is mostly linked to NCI Thesaurus (NCIt) and NCI Metathesaurus (NCIm) codes.
<h3 id="UserProfilesBroaderCommunity-AmericanCollegeofCardiology(ACC)">
  American College of Cardiology (ACC)
</h3>
American College of Cardiology (ACC) and EVS have worked together for several years on a number of projects, some also involving
<a href="https://wiki.nci.nih.gov/display/EVS/12+-+User+Profiles+-+NIH" rel="nofollow">
  NHLBI
</a>
,
<a href="https://wiki.nci.nih.gov/display/EVS/13+-+User+Profiles+-+Government+and+Standards+Organizations" rel="nofollow">
  CDISC
</a>
and Duke University. EVS worked with ACC and CDISC to develop 389 terms for the ACC/CDISC Cardiovascular Disease Therapeutic Area standard, bundled in 30 new codelists as well as a number of existing codelists. ACC made use of 157 existing NCI Thesaurus terms and added 232 new ones, while EVS added 194 ACC concept definitions as NCI preferred definitions.EVS Related References
<ol>
  <li>
    Anderson HV, Weintraub WS, Radford MJ, Kremers MS, Roe MT, Shaw RE, Pinchotti DM, Tcheng JE. Standardized Cardiovascular Data for Clinical Research, Registries, and Patient Care: A Report from the Data Standards Workgroup of the National Cardiovascular Research Infrastructure Project.  J Am Coll Cardiol. 2013 May 7;61(18):1835-46. doi: 10.1016/j.jacc.2012.12.047. Epub 2013 Mar 6. PubMed PMID: 23500238. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/23500238" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-AmericanSocietyofClinicalOncology(ASCO)">
  American Society of Clinical Oncology (ASCO)
</h3>
ASCO brings together more than 40,000 oncology health care professionals in many activities related to EVS, most notably CancerLinQ.
<h4 id="UserProfilesBroaderCommunity-CancerLearningIntelligenceNetworkforQuality(CancerLinQ)">
  Cancer Learning Intelligence Network for Quality (CancerLinQ)
</h4>
ASCO created (
<a href="https://cancerlinq.org/" rel="nofollow">
  CancerLinQ
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
) to provide a big data based learning health care platform for oncology, now encompassing more than 1,000,000 patient records. NCI Metathesaurus is used as the content knowledge source for processing and standardizing these data across EHR systems.EVS Related References
<ol>
  <li>
    Schilsky RL. Finding the Evidence in Real-World Evidence: Moving from Data to Information to Knowledge.  J Am Coll Surg. 2017 Jan;224(1):1-7. doi: 10.1016/j.jamcollsurg.2016.10.025. PubMed PMID: 27989954. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27989954" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Schilsky RL, Miller RS. Creating a Learning Health Care System in Oncology.  Oncology Informatics: Using Health Information Technology to Improve Processes and Outcomes in Cancer. Hesse BW, Ahern DK, Beckjord E, eds. Elsevier Academic Press (2016), pp. 3–21.
  </li>
  <li>
    Visvanathan K, Levit LA, Raghavan D, Hudis CA, Wong S, Dueck A, Lyman GH. Untapped Potential of Observational Research to Inform Clinical Decision Making: American Society of Clinical Oncology Research Statement.  J Clin Oncol. 2017 Mar 30:JCO2017726414. doi: 10.1200/JCO.2017.72.6414. [Epub ahead of print] PubMed PMID: 28358653. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28358653" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-CatalogueofSomaticMutationsinCancer(COSMIC)">
  Catalogue of Somatic Mutations in Cancer (COSMIC)
</h3>
<a href="http://cancer.sanger.ac.uk/cosmic" rel="nofollow">
  COSMIC
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
provides a very broad, high-resolution resource for exploring the impact of somatic mutations in human cancer. COSMIC has translated its custom tumor classification into NCI Thesaurus (NCIt), &#34;selected as the highest-resolution public ontology across cancer diseases.&#34; (Forbes et al. 2016).EVS Related References
<ol>
  <li>
    Forbes SA, Beare D, Boutselakis H, Bamford S, Bindal N, Tate J, Cole CG, Ward S, Dawson E, Ponting L, Stefancsik R, Harsha B, Kok CY, Jia M, Jubb H, Sondka Z, Thompson S, De T, Campbell PJ. COSMIC: somatic cancer genetics at high-resolution. Nucleic Acids Res. 2016 Nov 28. pii: gkw1121. [Epub ahead of print] PubMed PMID: 27899578. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27899578" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-ClinicalGenomeResource(ClinGen)">
  Clinical Genome Resource (ClinGen)
</h3>
<a href="https://www.clinicalgenome.org/" rel="nofollow">
  ClinGen
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
launched in 2013 as an NIH supported partnership among public, academic, and private institutions dedicated to building an authoritative central resource that defines the clinical relevance of genes and variants for use in precision medicine and research. The ClinGen Somatic Cancer Working Group developed with ClinVar and multiple stakeholders a consensus set of minimal variant level data, recommending that Cancer Type be coded using either NCI Thesaurus or Oncotree, which itself includes the NCI Thesaurus code.EVS Related References
<ol>
  <li>
    Ritter DI, Roychowdhury S, Roy A, Rao S, Landrum MJ, Sonkin D, Shekar M, Davis CF, Hart RK, Micheel C, Weaver M, Van Allen EM, Parsons DW, McLeod HL, Watson MS, Plon SE, Kulkarni S, Madhavan S; ClinGen Somatic Cancer Working Group. Somatic cancer variant curation and harmonization through consensus minimum variant level data.  Genome Med. 2016 Nov 4;8(1):117. PubMed PMID: 27814769; PubMed Central PMCID: PMC5095986. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27814769" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-DukeUniversity">
  Duke University
</h3>
Duke University and EVS have collaborated for several years on various projects. As one example, the Research Informatics group of the Duke Clinical Research Institute (DCRI) has used vocabulary services from the EVS LexEVS terminology server in adding standard metadata to the Cardiovascular DAM. Duke and EVS have worked together with other partners such as CDISC, NHLBI, and the American College of Cardiology, in the development of shared terminology and other standards involving clinical trials, case report forms, cardiology, tuberculosis, and other content.EVS Related References
<ol>
  <li>
    Anderson HV, Weintraub WS, Radford MJ, Kremers MS, Roe MT, Shaw RE, Pinchotti DM, Tcheng JE. Standardized Cardiovascular Data for Clinical Research, Registries, and Patient Care: A Report from the Data Standards Workgroup of the National Cardiovascular Research Infrastructure Project.  J Am Coll Cardiol. 2013 May 7;61(18):1835-46. doi: 10.1016/j.jacc.2012.12.047. Epub 2013 Mar 6. PubMed PMID: 23500238. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/23500238" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-EmoryUniversity">
  Emory University
</h3>
Emory University has deployed and extended several EVS resources. Emory is using LexEVS to develop and host local ontologies. Uses include terminology support for an analytic data warehouse, which incorporates custom patient classes defined using ICD-9 codes.EVS Related References
<ol>
  <li>
    Vergara-Niedermayr C, Wang F, Pan T, Kurc T, Saltz J. Semantically Interoperable XML Data.  Int. J. Semantic Computing. 2013 Sept;7(3). DOI: 10.1142/S1793351X13500037 [
    <a href="http://www.worldscientific.com/doi/abs/10.1142/S1793351X13500037" rel="nofollow">
      Online
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Vergara-Niedermayr C, Wang F, Pan T, Kurc T, Saltz J. Semantically Interoperable XML Data.  Emory University Center for Comprehensive Informatics Technical Report CCI-TR-2012-1, January 12, 2012. [
    <a href="http://confluence.cci.emory.edu:8090/download/attachments/4033450/CCI-TR-2012-1.pdf" rel="nofollow">
      Emory
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Zheng S, Wang F, Lu J, Saltz J. Enabling Ontology Based Semantic Queries in Biomedical Database Systems.  Emory University Center for Comprehensive Informatics Technical Report CCI-TR-2012-3, March 20, 2012. [
    <a href="http://confluence.cci.emory.edu:8090/download/attachments/4033450/CCI-TR-2012-3.pdf" rel="nofollow">
      Emory
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-EuropeanBioinformaticsInstitute(EBI)">
  European Bioinformatics Institute (EBI)
</h3>
The European Bioinformatics Institute
<a href="http://www.ebi.ac.uk/" rel="nofollow">
  (EBI
</a>
) Experimental Factor Ontology (
<a href="http://www.ebi.ac.uk/efo/" rel="nofollow">
  EFO
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
) provides a systematic description of many experimental variables, and is used to support a number of EBI databases as well as the National Human Genome Research Institute (NHGRI) Genome-Wide Association Study (
<a href="http://www.genome.gov/gwastudies/" rel="nofollow">
  GWAS
</a>
) catalogue. It combines parts of several biological ontologies, covering domains such as anatomy, disease and chemical compounds. The EFO reuses cancer-related terminology from the NCI Thesaurus (NCIt).EVS Related References
<ol>
  <li>
    Malone J, Rayner TF, Bradley XZ, Parkinson H. Developing an application focused experimental factor ontology: embracing the OBO Community.  Bio-Ontologies SIG, ISMB 2008 Toronto Canada, 20 July 2008. [
    <a href="http://www.ebi.ac.uk/efo/malone_bioontologies_2008.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-GeneralElectric(GE)">
  General Electric (GE)
</h3>
GE is developing a platform called Qualibria, which includes LexEVS as a terminology server. VKC has been working with GE since 2008. As part of the collaboration, GE created an extension to the LexEVS 5.1 API based on the Common Terminology Services specification. GE made the code for that extension available to the community via the VKC web site, and bundled EVS-supported open source technology into GE&#39;s commercial healthcare product.
<h3 id="UserProfilesBroaderCommunity-GeorgetownUniversity">
  Georgetown University
</h3>
Georgetown uses LexEVS and other EVS resources for its cancer Bench-to-Bedside (caB2B) project and other translational medicine activities. Both a local LexEVS installation and NCI&#39;s production LexEVS servers provide terminology support for this project.
<h3 id="UserProfilesBroaderCommunity-JacksonLaboratory">
  Jackson Laboratory
</h3>
The Jackson Laboratory has been a close partner with NCI – and EVS in particular – for over 10 years. They have been closely involved in the MMHCC and other initiatives described in the NCI Division of Cancer Biology (DCB) section of
<a href="https://wiki.nci.nih.gov/display/EVS/11+-+User+Profiles+-+NCI" title="11 - User Profiles - NCI" rel="nofollow">
  11 - User Profiles - NCI
</a>
. EVS publishes their Adult Mouse Anatomy (MA) on NCI servers and browsers (see
<a href="http://nciterms.nci.nih.gov/ncitbrowser/pages/vocabulary.jsf?dictionary=MA" rel="nofollow">
  NCI Term Browser
</a>
).As part of the Mouse-Human Anatomy Project (MHAP), anatomy terms in MA and the NCIt human anatomy were compared and harmonized, and a formal mapping was jointly created and validated in 2006; this mapping has been recently updated and published as a mapping accessible through the LexEVS server APIs and NCI Term Browser (see
<a href="http://nciterms.nci.nih.gov/ncitbrowser/pages/vocabulary.jsf?dictionary=MA_to_NCIt_Mapping" rel="nofollow">
  MA to NCIt Mapping
</a>
). As both anatomical ontologies are being used to annotate different types of research data for mouse and human, respectively, this cross-mapping between the two ontologies facilitates the integration of mouse and human data, and the translation of basic research discoveries into clinical settings.EVS Related References
<ol>
  <li>
    Bodenreider O, Hayamizu TF, Ringwald M, De Coronado S, Zhang S. Of mice and men: aligning mouse and human anatomies.  AMIA Annu Symp Proc. 2005:61-5. PubMed PMID: 16779002; PubMed Central PMCID: PMC1560846. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/16779002" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Hayamizu TF, de Coronado S, Fragoso G, Sioutos N, Kadin JA, Ringwald M. The mouse-human anatomy ontology mapping project.  Database (Oxford). 2012 Mar 20;2012:bar066. Print 2012. PubMed PMID: 22434834. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/22434834" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://database.oxfordjournals.org/content/2012/bar066.full" rel="nofollow">
      Full Text
    </a>
    ]
  </li>
  <li>
    Kogan SC, Ward JM, Anver MR, Berman JJ, Brayton C, Cardiff RD, Carter JS, de Coronado S, Downing JR, Fredrickson TN, Haines DC, Harris AW, Harris NL, Hiai H, Jaffe ES, MacLennan IC, Pandolfi PP, Pattengale PK, Perkins AS, Simpson RM, Tuttle MS, Wong JF, Morse HC 3rd; Hematopathology subcommittee of the Mouse Models of Human Cancers Consortium. Bethesda proposals for classification of nonlymphoid hematopoietic neoplasms in mice.  Blood. 2002 Jul 1;100(1):238-45. PubMed PMID: 12070033. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/12070033" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-MayoClinic">
  Mayo Clinic
</h3>
Mayo Clinic has partnered with, supported and used EVS resources in a variety of ways for more than 10 years. This has included joint work on analyzing cancer clinical trials vocabulary and informatics needs in the U.S., improving research and clinical data representation and reuse through projects such as PHONT and
<a href="https://wiki.nci.nih.gov/display/EVS/14+-+User+Profiles+-+Broader+Community#id-14-UserProfiles-BroaderCommunity-StrategicHealthITAdvancedResearchProjectsArea4-SecondaryUseofEHRDataSHARPn" rel="nofollow">
  SHARPn
</a>
, and development of shared community standards for vocabulary representation and infrastructure including the draft HL7/OMG standard
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=63996154" rel="nofollow">
  Common Terminology Services Release 2 (CTS 2) specification
</a>
and the the Mayo Clinic LexEVS terminology server that EVS has supported and uses. Mayo Clinic has a lead role in the ICD-11 revision effort, and Mayo has worked with EVS to leverage NCI Thesaurus cancer content and EVS subject matter experts in supporting revisions to the WHO ICD terminologies (see the
<a href="https://wiki.nci.nih.gov/display/EVS/13+-+User+Profiles+-+Government+and+Standards+Organizations#13-UserProfiles-GovernmentandStandardsOrganizations-WorldHealthOrganization%28WHO%29" rel="nofollow">
  WHO section
</a>
).
<h4 id="UserProfilesBroaderCommunity-PharmacogenomicsResearchNetwork(PGRN)OntologyNetworkResource(PHONT)">
  Pharmacogenomics Research Network (PGRN) Ontology Network Resource (PHONT)
</h4>
Mayo Clinic is the primary site for
<a href="http://informatics.mayo.edu/phont" rel="nofollow">
  PHONT
</a>
, a networked PGRN ontology resource that has been one of the highest-volume users of EVS resources. PHONT supports clear annotation and representation of phenotype (disease, adverse event, or clinical and physiological outcomes) to support data integration and cross-database analyses. PHONT has deployed its own instance of LexEVS, depending in particular on full support of CTS 2 value sets. PHONT is a collaboration with Case Western Reserve University, Harvard Medical School, MD Anderson Cancer Center, Memorial Sloan Kettering Cancer Center, University of Erlangen, and Washington University.
<h4 id="UserProfilesBroaderCommunity-StrategicHealthITAdvancedResearchProjects:Area4-SecondaryUseofEHRData(SHARPn)">
  Strategic Health IT Advanced Research Projects: Area 4 - Secondary Use of EHR Data (SHARPn)
</h4>
<a href="http://sharpn.org" rel="nofollow">
  SHARPn
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
<a href="http://www.cancer.gov/global/web/policies/exit" rel="nofollow"></a>
uses LexEVS and its value set support to help enable the reuse of EHR data for secondary purposes, such as clinical research and public health, as part of the Office of the National Coordinator for Health Information Technology SHARP Program.EVS Related References
<ol>
  <li>
    Chute CG, Carter JS, Tuttle MS, Haber MW, Brown SH. Integrating pharmacokinetics knowledge into a drug ontology as an extension to support pharmacogenomics.  AMIA Annu Symp Proc. 2003:170-4. PubMed PMID: 14728156; PubMed Central PMCID: PMC1480302. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/14728156" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmid/14728156/?tool=pubmed" rel="nofollow">
      Free PMC Article
    </a>
    ]
  </li>
  <li>
    Chute CG, Pathak J, Savova GK, Bailey KR, Schor MI, Hart LA, Beebe CE, Huff SM. The SHARPn project on secondary use of Electronic Medical Record data: progress, plans, and possibilities.  AMIA Annu Symp Proc. 2011;2011:248-56. Epub 2011 Oct 22. PubMed PMID: 22195076; PubMed Central PMCID: PMC3243296. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/22195076" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Gwaltney K, Chute C, Hageman D, Kibbe W, McCormick K, Reeves D, Wright L. An assessment of cancer clinical trials vocabulary and IT infrastructure in the U.S.  Proc AMIA Symp. 2001:224-8. PubMed PMID: 11825185; PubMed Central PMCID: PMC2243595. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/11825185" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmid/11825185/?tool=pubmed" rel="nofollow">
      Free PMC Article
    </a>
    ]
  </li>
  <li>
    Jiang G, Solbrig HR, Iberson-Hurst D, Kush RD, Chute CG. A Collaborative Framework for Representation and Harmonization of Clinical Study Data Elements Using Semantic MediaWiki.  AMIA Summits Transl Sci Proc. 2010 Mar 1;2010:11-5. PubMed PMID: 21347136; PubMed Central PMCID: PMC3041544. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21347136" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Liu K, Chapman WW, Savova G, Chute CG, Sioutos N, Crowley RS. Effectiveness of Lexico-syntactic Pattern Matching for Ontology Enrichment with Clinical Documents.  Methods Inf Med. 2011;50(5):397-407. Epub 2010 Nov 8. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21057720" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Pathak J, Wang J, Kashyap S, Basford M, Li R, Masys DR, Chute CG. Mapping clinical phenotype data elements to standardized metadata repositories and controlled terminologies: the eMERGE Network experience.  J Am Med Inform Assoc. 2011 Jul-Aug;18(4):376-86. Epub 2011 May 19. PubMed PMID: 21597104; PubMed Central PMCID: PMC3128396. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21597104" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Zhu Q, Freimuth RR, Lian Z, Bauer S, Pathak J, Tao C, Durski MJ, Chute CG. Harmonization and semantic annotation of data dictionaries from the Pharmacogenomics Research Network: A case study.  J Biomed Inform. 2013 Apr;46(2):286-93. doi: 10.1016/j.jbi.2012.11.004. Epub 2012 Nov 29. PubMed PMID: 23201637; PubMed Central PMCID: PMC3606279. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/23201637" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    See also: LexEVS section of
    <a href="https://wiki.nci.nih.gov/display/EVS/Bibliography+on+EVS+and+Its+Use" title="Bibliography on EVS and Its Use" rel="nofollow">
      Bibliography on EVS and Its Use
    </a>
    .
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-MDAnderson">
  MD Anderson
</h3>
MD Anderson is using a wide range of EVS terminology content and technologies as part of its enterprise infrastructure. A LexEVS server and EVS terminology browsers have been deployed locally at MD Anderson. NCIt and other EVS terminology resources are also being used.
<h3 id="UserProfilesBroaderCommunity-MemorialSloanKetteringCancerCenter">
  Memorial Sloan Kettering Cancer Center
</h3>
The
<a href="http://mskcc.org/" rel="nofollow">
  MSK Cancer Center (MSK)
</a>
has used NCI Thesaurus (NCIt) in its precision oncology efforts, most notably through the OncoTree tumor type tree used in cBioPortal and its new OncoKB project.
<h4 id="UserProfilesBroaderCommunity-OncoKB">
  OncoKB
</h4>
<a href="http://oncokb.org/#/" rel="nofollow">
  OncoKB
</a>
was first released in June 2016 as a precision oncology knowledge base for annotation of somatic mutations in cancer. It contains information about the effects and treatment implications of specific cancer gene alterations. It is developed and maintained by the Knowledge Systems group in the Marie Josée and Henry R. Kravis Center for Molecular Oncology (CMO), which is integrating OncoKB information with cBioPortal.
<h4 id="UserProfilesBroaderCommunity-cBioPortal">
  cBioPortal
</h4>
The
<a href="http://www.cbioportal.org/" rel="nofollow">
  cBioPortal for Cancer Genomics
</a>
provides visualization, analysis, and download of large-scale cancer genomics data sets including
<a href="https://cancergenome.nih.gov/" rel="nofollow">
  The Cancer Genome Atlas (TCGA)
</a>
.  Originally developed by, and still hosted at, MSK, cBioPortal is now developed and maintained by a multi-institutional team.
<h4 id="UserProfilesBroaderCommunity-OncoTree">
  OncoTree
</h4>
The
<a href="http://oncotree.mskcc.org/oncotree/" rel="nofollow">
  OncoTree
</a>
CMO Tumor Type Tree provides a user-friendly interface to 524 tumor types from 32 tissues, most linked to NCI Thesaurus (NCIt) and NCI Metathesaurus (NCIm) codes.
<h3 id="UserProfilesBroaderCommunity-OhioStateUniversityMedicalCenter">
  Ohio State University Medical Center
</h3>
Ohio State is using LexEVS, NCIt, and NCIm, notably for its openMDR project. Ohio State University launched openMDR (open metadata repository) in 2009, using local instances of LexEVS, BioPortal, and caDSR.
<h3 id="UserProfilesBroaderCommunity-SeoulNationalUniversity,Korea">
  Seoul National University, Korea
</h3>
The Biomedical Knowledge Engineering (BiKE) lab adopted the 2005 version of the LexGrid model and over the last several years created an entire terminology-based application suite on that model called LexCare Suite. The VKC facilitated the signing of an agreement between Mayo Clinic and SNU&#39;s BiKE to solidify a collaboration under which they will work together on conferences and papers surrounding terminology creation/mapping/use. The BiKE mapping tool is of particular interest for a community tool in this regard. This would be a potentially significant contribution, as it would add a new tool in an area with a known gap in the current functionality of LexEVS.
<h3 id="UserProfilesBroaderCommunity-StanfordUniversity">
  Stanford University
</h3>
Since 2003, EVS has worked closely with the Stanford University Center for Biomedical Informatics Research (BMIR) and the NCBO project to develop shared community tools, standards and resources. Protégé OWL and NCI Protégé have pushed the envelope on open source software for ontology development and production management. NCBO and EVS collaborated on terminology metadata standards, with caBIG and the UK National Cancer Research Institute. NCBO hosts copies of NCI Thesaurus (NCIt), one of its most highly accessed terminologies. NCIt is also used by the NCBO Annotator tool for annotating documents and data with terminology concepts. NCBO and EVS have complementary resources, and share knowledge and expertise.
<h4 id="UserProfilesBroaderCommunity-Protégé">
  Protégé
</h4>
EVS has worked closely with BMIR and its predecessor, Stanford Medical Informatics. An early focus was development of Protégé OWL, so that EVS could move from proprietary to open source terminology editing software that used the emerging OWL DL standard; this was followed by support for an NCI specific plug-in that enable NCI terminology production management. Over time, many of these NCI specific changes have been rolled into the main Protégé 3.4 code, so that users worldwide could take advantage of such features as distributed collaborative terminology development. In addition, the NCI specific Protégé configuration, with its additional plug-ins, is available for non-NCI users. EVS editors and staff have been heavy users of Protégé, collaborators on collaborative ontology workflows, and testers of collaborative Protégé, Web Protégé and NCBO tools and services such as concept merging and terminology mapping tools.
<h4 id="UserProfilesBroaderCommunity-NationalCenterforBiomedicalOntology(NCBO)">
  National Center for Biomedical Ontology (NCBO)
</h4>
<a href="http://www.bioontology.org/" rel="nofollow">
  NCBO
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
bases its BioPortal terminology services on LexEVS, supporting a very wide collection of some 300 terminologies and ontologies that are actively used in biomedicine. BioPortal hosts copies of NCIt, which is generally at or near the top of the NCBO ontology use chart (see section 3 above). NCBO also makes its terminologies available through caGrid.
<h4 id="UserProfilesBroaderCommunity-TerminologyMetadata">
  Terminology Metadata
</h4>
NCBO and EVS have collaborated on standards for terminology metadata with caBIG participants and the UK National Cancer Research Institute (NCRI), to develop and promote standards for annotating the content, structure and use of ontologies and terminologies, based on earlier work of Stanford and a European group on the Ontology Metadata Vocabulary (OMV). This work has fed into the terminology metadata content now embedded in the CTS 2 (Common Terminology Services 2) standard published by HL7 and the OMG (Object Management Group).
<h4 id="UserProfilesBroaderCommunity-TissueMicroarrayDatabase(TMAD)">
  Tissue Microarray Database (TMAD)
</h4>
<a href="http://tma.im/" rel="nofollow">
  TMAD
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is an important public resource for raw and processed data (with stained tissue images) from tissue microarray experiments. TMAD uses NCI Thesaurus to index, browse and search tissues, and provides methods for data retrieval, grouping of data, analysis and visualization as well as export to standard formats.EVS Related References
<ol>
  <li>
    Bail S, Horridge M, Parsia B, Sattler U. The justificatory structure of the NCBO bioportal ontologies.  The Semantic Web–ISWC 2011. Springer 2011 pp.67-82. [
    <a href="http://iswc2011.semanticweb.org/fileadmin/iswc/Papers/Research_Paper/10/70310065.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Falconer SM, Tudorache T, Noy NF. An Analysis of Collaborative Patterns in Large-Scale Ontology Development Projects.  Proceedings of the sixth international conference on Knowledge capture (K-CAP &#39;11). 2011 ACM, New York, NY, USA. [
    <a href="http://dl.acm.org/citation.cfm?id=1999682" rel="nofollow">
      ACM
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ] [
    <a href="http://web.stanford.edu/%7Enatalya/papers/p25-falconer.pdf" rel="nofollow">
      Stanford
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Ghazvinian A, Noy NF, Musen MA. From mappings to modules: Using mappings to identify domain-specific modules in large ontologies.  KCAP 2011 - Proceedings of the 2011 Knowledge Capture Conference, 2011, pp. 33-40. [
    <a href="http://www.scopus.com/inward/record.url?eid=2-s2.0-79960276657&amp;partnerID=40&amp;md5=89ee8ee5dc29c216c12b55745959a1c4" rel="nofollow">
      Scopus
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Jonquet C, LePendu P, Falconer S, Coulet A, Noy NF, Musen MA, Shah NH NCBO Resource Index: Ontology-based search and mining of biomedical resources  Web Semantics: Science, Services and Agents on the World Wide Web, Volume 9, Issue 3, September 2011, Pages 316-324, ISSN 1570-8268, 10.1016/j.websem.2011.06.005. [
    <a href="http://www.sciencedirect.com/science/article/pii/S1570826811000485" rel="nofollow">
      ScienceDirect
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Jonquet C, Musen MA, Shah NH. Building a biomedical ontology recommender web service.  J Biomed Semantics. 2010 Jun 22;1 Suppl 1:S1. PubMed PMID: 20626921; PubMed Central PMCID: PMC2903720. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/20626921" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.jbiomedsem.com/content/pdf/2041-1480-1-S1-S1.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Jonquet C, Shah NH, Musen MA. Prototyping a Biomedical Ontology Recommender Service.  Bio-Ontologies: Knowledge in Biology, Stockholm, Sweden (2009).
  </li>
  <li>
    Jonquet C, Shah NH, Musen MA. The open biomedical annotator.  Summit on Translat Bioinforma. 2009 Mar 1;2009:56-60. PubMed PMID: 21347171; PubMed Central PMCID: PMC3041576. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21347171" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041576/?tool=pubmed" rel="nofollow">
      Full Text
    </a>
    ]
  </li>
  <li>
    Noy NF, Alexander PR, Harpaz R, Whetzel PL, Fergerson RW, Musen MA. Getting Lucky in Ontology Search: A Data-Driven Evaluation Framework for Ontology Ranking.  Proceedings of the 12th International Semantic Web Conference (ISWC 2013), Sydney, Australia, 21-25 Oct 2013. [
    <a href="http://www.stanford.edu/%7Enatalya/papers/iswc2013_bpSearch.pdf" rel="nofollow">
      PDF
    </a>
    ]
  </li>
  <li>
    Parai GK, Jonquet C, Xu R, Musen MA, Shah NH. The Lexicon Builder Web service: Building Custom Lexicons from two hundred Biomedical Ontologies.  AMIA Annu Symp Proc. 2010 Nov 13;2010:587-91. PubMed PMID: 21347046; PubMed Central PMCID: PMC3041331. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21347046" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041331/pdf/amia-2010_sympproc_0587.pdf" rel="nofollow">
      PDF
    </a>
    ]
  </li>
  <li>
    Rubin DL, Shah NH, Noy NF. Biomedical ontologies: a functional perspective.  Brief Bioinform. 2008 Jan;9(1):75-90. Epub 2007 Dec 12. Review. PubMed PMID: 18077472. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/18077472" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Salvadores M, Alexander PR, Musen MA, Noy NF. BioPortal as a Dataset of Linked Biomedical Ontologies and Terminologies in RDF.  Semantic Web Journal 2013; 4(3):277-284. Available online 31 Oct 2012. [
    <a href="http://www.semantic-web-journal.net/sites/default/files/swj278_4.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ] [
    <a href="http://iospress.metapress.com/content/4mut1t3p76662806/?p=554fa16e20074e95aa8c251b47a4faa3&amp;pi=0" rel="nofollow">
      Online
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Salvadores M, Alexander PR, Musen MA, Noy NF. The Quad Economy of a Semantic Web Ontology Repository.  The 7th International Workshop on Scalable Semantic Web Knowledge Base Systems (SSWS 2011). pp.14-29. [
    <a href="http://iswc2011.semanticweb.org/fileadmin/iswc/Papers/Workshops/SSWS/Salvadores-et-all-SSWS2011.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Sebastian A, Noy NF, Tudorache T, Musen MA. A generic ontology for collaborative ontology-development workflows.  Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics), 5268 LNAI, 2008, pp. 318-328. [
    <a href="http://www.scopus.com/inward/record.url?eid=2-s2.0-56649105039&amp;partnerID=40&amp;md5=3ee1963c5952f19952e28d3e049f3dd4" rel="nofollow">
      Scopus
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Shah NH, Bhatia N, Jonquet C, Rubin D, Chiang AP, Musen MA. Comparison of concept recognizers for building the Open Biomedical Annotator.  BMC Bioinformatics. 2009 Sep 17;10 Suppl 9:S14. PubMed PMID: 19761568; PubMed Central PMCID: PMC2745685. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/19761568" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.biomedcentral.com/1471-2105/10/S9/S14" rel="nofollow">
      Full Text
    </a>
    ]
  </li>
  <li>
    Shah NH, Jonquet C, Chiang AP, Butte AJ, Chen R, Musen MA. Ontology-driven indexing of public datasets for translational bioinformatics.  BMC Bioinformatics. 2009 Feb 5;10 Suppl 2:S1. PubMed PMID: 19208184; PubMed Central PMCID: PMC2646250. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/19208184" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Shah NH, Musen MA. UMLS-Query: a perl module for querying the UMLS.  AMIA Annu Symp Proc. 2008 Nov 6:652-6. PubMed PMID: 18998805; PubMed Central PMCID: PMC2656020. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/18998805" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2656020/?tool=pubmed" rel="nofollow">
      Full Text
    </a>
    ]
  </li>
  <li>
    Shah NH, Rubin DL, Espinosa I, Montgomery K, Musen MA. Annotation and query of tissue microarray data using the NCI Thesaurus.  BMC Bioinformatics. 2007 Aug 8;8:296. PubMed PMID: 17686183; PubMed Central PMCID: PMC1988837. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/17686183" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Strohmaier M, Walk S, Pöschko J, Lamprecht D, Tudorache T, Nyulas C, Musen MA, Noy NF. How Ontologies are Made: Studying the Hidden Social Dynamics Behind Collaborative Ontology Engineering Projects.  Web Semantics: Science, Services and Agents on the World Wide Web, North America, April 2013. [
    <a href="http://mx1.websemanticsjournal.org/index.php/ps/article/download/333/341" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Wu ST, Liu H, Li D, Tao C, Musen MA, Chute CG, Shah NH. Unified Medical Language System term occurrences in clinical notes: a large-scale corpus analysis.  J Am Med Inform Assoc. 2012 Apr 4. [Epub ahead of print] PubMed PMID: 22493050. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/22493050" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-SwissInstituteofBioinformatics(SIB)">
  Swiss Institute of Bioinformatics (SIB)
</h3>
The
<a href="http://www.isb-sib.ch/" rel="nofollow">
  SIB
</a>
Computer Analysis and Laboratory Investigation of Proteins of Human Origin (CALIPHO) group develops a number of biocurated resources including neXtProt and the Cellosaurus. Both resources make use of the NCI Thesaurus (NCIt).
<h4 id="UserProfilesBroaderCommunity-neXtProt">
  neXtProt
</h4>
<a href="http://www.nextprot.org/" rel="nofollow">
  neXtProt
</a>
maps Catalogue of Somatic Mutations in Cancer (COSMIC) cancer terms to the corresponding terms in NCIt so as to present to its users a standardized vocabulary for the provenance of cancer protein variations extracted from COSMIC.
<h4 id="UserProfilesBroaderCommunity-Cellosaurus">
  Cellosaurus
</h4>
The
<a href="http://web.expasy.org/cellosaurus" rel="nofollow">
  Cellosaurus
</a>
, a cell line thesaurus, uses NCIt to annotate cell lines originating from diseased patients and animals (mainly cancers of human and animal model origins and genetic diseases); as of March 2016, Cellosaurus contains over 25,000 cell lines linking to over 1,000 diseases in NCIt.EVS Related References
<ol>
  <li>
    Gaudet P, Michel PA, Zahn-Zabal M, Cusin I, Duek PD, Evalet O, Gateau A, Gleizes A, Pereira M, Teixeira D, Zhang Y, Lane L, Bairoch A. The neXtProt knowledgebase on human proteins: current status.  Nucleic Acids Res. 2015 Jan 28;43(Database issue):D764-70. doi: 10.1093/nar/gku1178. PubMed PMID: 25593349. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/25593349" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Mottin L, Gobeill J, Pasche E, Michel PA, Cusin I, Gaudet P, Ruch P. neXtA5: accelerating annotation of articles via automated approaches in neXtProt.  Database (Oxford). 2016 Jul 3;2016. pii: baw098. doi: 10.1093/database/baw098. PubMed PMID: 27374119; PubMed Central PMCID: PMC4930835. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27374119" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-UniversityofPittsburgh">
  University of Pittsburgh
</h3>
EVS resources, notably NCI Thesaurus (NCIt) and NCI Metathesaurus (NCIm), are now deeply rooted in University of Pittsburgh core informatics applications used by hundreds of basic, translational and clinical researchers, and by many more using those applications at other institutions. University of Pittsburgh uses the NCIt cancer, anatomy, and pathology findings terminologies for their research and informatics projects. Millions of pathology reports have been encoded using NCIm and indexed using NCIt. Collaborations are ongoing, and have resulted in several journal articles published over the last 8 years.
<h4 id="UserProfilesBroaderCommunity-OntologyDevelopmentInformationExtraction(ODIE)">
  Ontology Development Information Extraction (ODIE)
</h4>
<a href="http://www.bioontology.org/ODIE" rel="nofollow">
  ODIE
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is a software toolkit that uses ontologies, including NCIt, to perform information extraction tasks from clinical documents and uses clinical documents to enhance existing ontologies. ODIE is used by the University of Pittsburgh, University of California, San Diego, and Children&#39;s Hospital, Boston.Multiple informatics projects at Pittsburgh build on the ODIE framework.
<h4 id="UserProfilesBroaderCommunity-TextInformationExtractionSystem(TIES)">
  Text Information Extraction System (TIES)
</h4>
<a href="http://ties.dbmi.pitt.edu/" rel="nofollow">
  TIES
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is an end-to-end Natural Language Processing pipeline and clinical document search engine, which started as caTIES with a focus on encoding pathology reports to support access to human tissues. TIES uses NCIm as the source of concepts for named entity recognition. It also relies on NCIt for ontology-based inference, indexing and retrieval. TIES is used by multiple investigators and deployed at cancer centers and other organizations including Roswell Park, University of Arkansas, and University of Pennsylvania. At University of Pittsburgh, the system currently supports more than 200 researchers.EVS Related References
<ol>
  <li>
    Amin W, Kang HP, Becich MJ. Data Management, Databases, and Warehousing.  Biomedical Informatics for Cancer Research. (M.F. Ochs, J.T. Casagrande, R.V. Davuluri, eds). DOI 10.1007/978-1-4419-5714-6_3, © Springer Science+Business Media, LLC 2010. pp.39-71.
  </li>
  <li>
    Crowley RS, Castine M, Mitchell K, Chavan G, McSherry T, Feldman M. caTIES: a grid based system for coding and retrieval of surgical pathology reports and tissue specimens in support of translational research.  J Am Med Inform Assoc. 2010 May-Jun;17(3):253-64. PubMed PMID: 20442142; PubMed Central PMCID: PMC2995710. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/20442142" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://jamia.bmj.com/cgi/pmidlookup?view=long&amp;pmid=20442142" rel="nofollow">
      Full Text
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Crowley RS, Tseytlin E, Jukic D. ReportTutor - an intelligent tutoring system that uses a natural language interface.  AMIA Annu Symp Proc. 2005:171-5. PubMed PMID: 16779024; PubMed Central PMCID: PMC1560511. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/16779024" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Jacobson RS, Becich MJ, Bollag RJ, Chavan G, Corrigan J, Dhir R, Feldman MD, Gaudioso C, Legowski E, Maihle NJ, Mitchell K, Murphy M, Sakthivel M, Tseytlin E, Weaver J. A Federated Network for Translational Cancer Research Using Clinical Data and Biospecimens.  Cancer Res. 2015 Dec 15;75(24):5194-201. doi:10.1158/0008-5472.CAN-15-1973. Review. PubMed PMID: 26670560; PubMed Central PMCID: PMC4683415.[
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/26670560" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Kang HP, Borromeo CD, Berman JJ, Becich MJ. The tissue microarray OWL schema: An open-source tool for sharing tissue microarray data.  J Pathol Inform. 2010 Jul 13;1. pii: 9. PubMed PMID: 20805954; PubMed Central PMCID: PMC2929536. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/20805954" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Liu K, Chapman WW, Savova G, Chute CG, Sioutos N, Crowley RS. Effectiveness of Lexico-syntactic Pattern Matching for Ontology Enrichment with Clinical Documents.  Methods Inf Med. 2011;50(5):397-407. Epub 2010 Nov 8. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21057720" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Tobias J, Chilukuri R, Komatsoulis GA, Mohanty S, Sioutos N, Warzel DB, Wright LW, Crowley RS. The CAP Cancer Protocols – A Case Study of caCORE Based Data Standards Implementation to Integrate with the Cancer Biomedical Informatics Grid.  BMC Medical Informatics Decision Making, 20; 6:25, 2006. [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1524939/?tool=pubmed" rel="nofollow">
      Free PMC Article
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-WashingtonUniversity">
  Washington University
</h3>
Washington University uses LexEVS and EVS terminology content in its clinical data warehouse project (CIDER). Washington University deployed LexEVS in 2008 as the terminology server for CIDER. The terminologies are used to code data and for information retrieval, supporting both research and clinical enterprise infrastructure.
<h3 id="UserProfilesBroaderCommunity-YaleUniversity">
  Yale University
</h3>
Yale University has used EVS resources in several biomedical research and informatics projects, including some with a special focus on semantic web technologies.EVS Related References
<ol>
  <li>
    McCusker JP, Phillips JA, González Beltrán A, Finkelstein A, Krauthammer M. Semantic web data warehousing for caGrid.  BMC Bioinformatics. 2009 Oct 1;10 Suppl 10:S2. PubMed PMID: 19796399; PubMed Central PMCID: PMC2755823. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/19796399" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Shifman MA, Li Y, Colangelo CM, Stone KL, Wu TL, Cheung KH, Miller PL, Williams KR. YPED: a web-accessible database system for protein expression analysis.  J Proteome Res. 2007 Oct;6(10):4019-24. Epub 2007 Sep 15. PubMed PMID: 17867667. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/17867667" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h2 id="UserProfilesBroaderCommunity-ProjectandTopical">
  Project and Topical
</h2>
<h3 id="UserProfilesBroaderCommunity-AnimalModelsandMappings">
  Animal Models and Mappings
</h3>
EVS has worked since 1999 on a variety of animal models of cancer and terminology mappings between human and non-human species. In addition to work described earlier involving the NCI Division of Cancer Biology with extensive community outreach (see the
<a href="https://wiki.nci.nih.gov/display/EVS/11+-+User+Profiles+-+NCI#11-UserProfiles-NCI-NCIDivisionofCancerBiology%28DCB%29" rel="nofollow">
  detailed profile
</a>
), and the section above on work with
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694982#14-UserProfiles-Community-JacksonLaboratory" rel="nofollow">
  the Jackson Laboratory
</a>
, EVS has been involved in a number of other community efforts.EVS has extended animal model terminology support to cover rats and zebrafish using two important community standards:
<ul>
  <li>
    International Harmonization of Rat Nomenclature (RENI) was used as the foundation for the Terminology of Rat Pathologic Diagnoses in NCI Thesaurus (NCIt).
  </li>
  <li>
    Zebrafish Information Network (ZFIN) zebrafish anatomy is provided as a standalone terminology in EVS systems including the
    <a href="http://nciterms.nci.nih.gov/ncitbrowser/pages/vocabulary.jsf?dictionary=Zebrafish" rel="nofollow">
      NCI Term Browser
    </a>
    .
  </li>
</ul>
EVS terminology is also being used in community efforts such as those below.
<h4 id="UserProfilesBroaderCommunity-Uberon">
  Uberon
</h4>
<a href="http://uberon.org/" rel="nofollow">
  Uberon
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is an integrated cross-species anatomy ontology constructed using a combination of semi-automated methods and manual curation. The ontology consists of classes representing anatomical entities that are shared across a variety of metazoan organisms, with a heavy bias towards model organisms and human anatomy. Uberon contains extensive cross-references between its terms and other anatomy ontologies, and draws heavily on NCI Thesaurus (NCIt) as well as the Adult Mouse Anatomy (MA) ontology (part of the caBIG-funded Mouse-Human Anatomy Project, MHAP).
<h4 id="UserProfilesBroaderCommunity-OntologyAlignmentEvaluationInitiative(OAEI)">
  Ontology Alignment Evaluation Initiative (OAEI)
</h4>
The
<a href="http://oaei.ontologymatching.org/" rel="nofollow">
  OAEI
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is a collaborative effort in the ontology alignment community aimed at rigorous and extensive evaluation of ontology alignment technologies. Since 2007, the OAEI has used the mouse-human anatomy set, with some modifications, as a “gold standard mapping” example of a “real world case” in an annual competitive evaluation of ontology matching approaches.
<h3 id="UserProfilesBroaderCommunity-CommonBiorepositoryModel(CBM)">
  Common Biorepository Model (CBM)
</h3>
EVS has provided about 1,300 concepts for this model through support for caDSR. CBM is used in many domains including clinical trials management, ICR, in vivo imaging, and tissue banks and pathology tools. In early 2012, EVS provided support for mapping SNOMED concepts used in caTissue with NCIt concepts used in CBM, to facilitate data sharing.
<h3 id="UserProfilesBroaderCommunity-eMERGENetwork">
  eMERGE Network
</h3>
The
<a href="http://www.gwas.net" rel="nofollow">
  eMERGE (electronic MEdical Records and GEnomics) Network
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
was initiated by National Human Genome Research Institute (NHGRI) in 2007 as a national research consortium to develop, disseminate, and apply research methods that combine DNA biorepositories with electronic medical record (EMR) systems for large-scale, high-throughput genetic research. EVS terminology plays an important role these efforts, notably through use in the
<a href="https://victr.vanderbilt.edu/eleMAP/" rel="nofollow">
  eleMAP tool
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
developed at Vanderbilt to help researchers harmonize their local phenotype data dictionaries to existing metadata and terminology standards.EVS Related References
<ol>
  <li>
    Pathak J, Wang J, Kashyap S, Basford M, Li R, Masys DR, Chute CG. Mapping clinical phenotype data elements to standardized metadata repositories and controlled terminologies: the eMERGE Network experience.  J Am Med Inform Assoc. 2011 Jul-Aug;18(4):376-86. Epub 2011 May 19. PubMed PMID: 21597104; PubMed Central PMCID: PMC3128396. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21597104" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-eTOX">
  eTOX
</h3>
<a href="http://www.e-tox.net/" rel="nofollow">
  eTOX
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is a European consortium established in 2010 by universities, pharmaceutical and biotech companies to share and use toxicology data. It is a pre-competitive collaboration whose main goal is to create and distribute tools to predict drug side-effects based on pre-clinical experiments. Aims are a better in silico predictability of potential adverse events and reducing the use of animals in toxicological research. The NCI Thesaurus is one of several vocabularies/ontologies used in eTOX to support shared semantics, principally the terms associated with entities as well as the relations between terms (see
<a href="http://bioportal.bioontology.org/projects/eTOX" rel="nofollow">
  BioPortal
</a>
). eTOX is funded by the Innovative Medicines Initiative (IMI), whose web site provides additional information (see
<a href="http://www.imi.europa.eu/content/etox" rel="nofollow">
  IMI eTox
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
).
<h3 id="UserProfilesBroaderCommunity-GlobalAlignmentofImmunizationsafetyAssessmentinpregnancy(GAIA)">
  Global Alignment of Immunization safety Assessment in pregnancy (GAIA)
</h3>
The Global Alignment of Immunization safety Assessment in pregnancy (
<a href="http://gaia-consortium.net/" rel="nofollow">
  GAIA
</a>
) is a global consortium to develop common standards, guidance, and tools to strengthen programs of immunization in pregnancy, with a specific focus on low and middle income countries. GAIA seeks to improve data and understanding on maternal, pregnancy, fetal, and neonatal health outcome assessment.The National Institute of Child Health and Human Development (
<a href="http://www.nichd.nih.gov/" rel="nofollow">
  NICHD
</a>
) and EVS have been collaborating with GAIA since 2014 to help develop standard terminology related to Fetal and Neonatal Events, Maternal, and Pregnancy Events and Outcomes. This expands on the
<a href="http://www.cancer.gov/research/resources/terminology/pediatric" rel="nofollow">
  Pediatric Terminology
</a>
joint effort, ongoing since 2009, to establish a core library of harmonized pediatric terms that enable clinical investigators to more readily compare and aggregate data across clinical research portfolios. To this end, terminology data and draft standards were developed as part of the NCI Thesaurus (NCIt) by experts from the NICHD, EVS, and other participants. All NCIt content is freely available without restriction.EVS Related References
<ol>
  <li>
    Bonhoeffer J, Kochhar S, Hirschfeld S, Heath PT, Jones CE, Bauwens J, Honrado Á, Heininger U, Muñoz FM, Eckert L, Steinhoff M, Black S, Padula M, Sturkenboom M, Buttery J, Pless R, Zuber P; GAIA project participants. Global alignment of immunization safety assessment in pregnancy - The GAIA project.  Vaccine. 2016 Dec 1;34(49):5993-5997. doi: 10.1016/j.vaccine.2016.07.006. PubMed PMID: 27751641. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27751641" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-HumanStudiesDatabase(HSDB)">
  Human Studies Database (HSDB)
</h3>
<a href="http://hsdbwiki.org/" rel="nofollow">
  HSDB
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is a consortium of research institutions defining and implementing a shared informatics infrastructure covering both interventional and observational human studies. HSDB is using LexEVS as a core component of its collaborative, distributed, clinical research systems. The
<a href="http://code.google.com/p/ontology-of-clinical-research/" rel="nofollow">
  OCRe Terminology
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
is served through an NCBO site, which uses LexEVS.EVS Related References
<ol>
  <li>
    Sim I, Carini S, Tu S, Wynden R, Pollock BH, Mollah SA, Gabriel D, Hagler HK, Scheuermann RH, Lehmann HP, Wittkowski KM, Nahm M, Bakken S. The Human Studies Database Project: Federating Human Studies Design Data Using the Ontology of Clinical Research.  AMIA Summits Transl Sci Proc. 2010: 51–55. [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3041546/" rel="nofollow">
      Online
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-InternationalHumanEpigenomeConsortium(IHEC)">
  International Human Epigenome Consortium (IHEC)
</h3>
<a href="http://ihec-epigenomes.org/" rel="nofollow">
  IHEC
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
started in 2010 as a global consortium with the primary goal of providing free access to high-resolution reference human epigenome maps for normal and disease cell types to the research community. IHEC coordinates the production of reference epigenome maps through the characterization of the regulome, methylome, and transcriptome from a wide range of tissues and cell types. IHEC chose NCIt as the ontology to represent disease information within the Consortium, which also uses NCI Metathesaurus as a key resource.EVS Related References
<ol>
  <li>
    Bujold D, Morais DA, Gauthier C, Côté C, Caron M, Kwan T, Chen KC, Laperle J, Markovits AN, Pastinen T, Caron B, Veilleux A, Jacques PÉ, Bourque G. The International Human Epigenome Consortium Data Portal.  Cell Syst. 2016 Nov 23;3(5):496-499.e2. doi: 10.1016/j.cels.2016.10.019. PubMed PMID: 27863956. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27863956" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Dyke SO, Cheung WA, Joly Y, Ammerpohl O, Lutsik P, Rothstein MA, Caron M, Busche S, Bourque G, Rönnblom L, Flicek P, Beck S, Hirst M, Stunnenberg H, Siebert R, Walter J, Pastinen T. Epigenome data release: a participant-centered approach to privacy protection.  Genome Biol. 2015 Jul 17;16(1):142. PubMed PMID: 26185018; PubMed Central PMCID: PMC4504083. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/26185018" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Fernández JM, de la Torre V, Richardson D, Royo R, Puiggròs M, Moncunill V, Fragkogianni S, Clarke L; BLUEPRINT Consortium., Flicek P, Rico D, Torrents D, Carrillo de Santa Pau E, Valencia A. The BLUEPRINT Data Analysis Portal.  Cell Syst. 2016 Nov 23;3(5):491-495.e5. doi: 10.1016/j.cels.2016.10.021. PubMed PMID: 27863955. [
    <a href="https://www.ncbi.nlm.nih.gov/pubmed/27863955" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-ImagingStandards">
  Imaging Standards
</h3>
Imaging standards development requires the support of multiple products made available through EVS, including publication of RadLex terminology, developed through a collaboration of the Radiological Society of North America (RSNA), which convened experts in imaging informatics and radiological subspecialties to create this resource, now made freely available. RadLex has developed into a rich, structured radiology-specific ontology, which currently includes more than 30,000 terms and to which EVS provides content as well as publication support. EVS incorporates RadLex into the NCI Metathesaurus, and also supports imaging terminology in the NCI Thesaurus as needed by the imaging community.EVS Related References
<ol>
  <li>
    Hazen R, Van Esbroeck AP, Mongkolwat P, Channin DS. Automatic Extraction of Concepts to Extend RadLex.  Journal of Digital Imaging 2011 Feb;24(1):165-169. [
    <a href="http://www.springerlink.com/content/p889312g07534661/" rel="nofollow">
      PDF
    </a>
    ]
  </li>
  <li>
    Lacson R, Andriole KP, Prevedello LM, Khorasani R. Information from Searching Content with an Ontology-Utilizing Toolkit (iSCOUT).  Journal of Digital Imaging 2012 Feb 14:1-8 [Epub ahead of print] PubMed PMID: 22349993. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/22349993" rel="nofollow">
      PubMed
    </a>
    ] [
    <a href="http://dx.doi.org/10.1007/s10278-012-9463-9" rel="nofollow">
      Online
    </a>
    ]
  </li>
  <li>
    Warden GI, Lacson R, Khorasani R. Leveraging Terminologies for Retrieval of Radiology Reports with Critical Imaging Findings.  AMIA Annu Symp Proc. 2011 Oct;2011:1481-8. Epub 2011 Oct 22. PubMed PMID: 22195212; PubMed Central PMCID: PMC3243125. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/22195212" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-LexEVSAdopterCommunity">
  LexEVS Adopter Community
</h3>
Many other organizations have adopted the LexEVS terminology server and related tools. Key examples described in the institutional portion of this section are:
<ol>
  <li>
    Emory University
  </li>
  <li>
    GE
  </li>
  <li>
    MD Anderson
  </li>
  <li>
    Mayo Clinic (incl. PHONT and SHARPn)
  </li>
  <li>
    McGill University Health Center, Canada
  </li>
  <li>
    Ohio State University Medical Center
  </li>
  <li>
    Seoul National University, Korea
  </li>
  <li>
    Stanford/National Center for Biomedical Ontology (NCBO)
  </li>
  <li>
    Washington University
  </li>
</ol>
For literature references, see the LexEVS section of
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=51413215" rel="nofollow">
  Bibliography on EVS and Its Use
</a>
.
<h3 id="UserProfilesBroaderCommunity-Nanotechnology">
  Nanotechnology
</h3>
Nanotechnology, and more specifically nanomedicine, has become important in the development of reagents for cancer detection, diagnosis and treatment. NCI established
<a href="http://nano.cancer.gov/action/programs/ccne.asp" rel="nofollow">
  Cancer Centers of Nanotechnology Excellence (CCNE)
</a>
to support translational nanomedical research, and collaborative efforts are supported by NHLBI, NHGRI, FDA and others. EVS provides a range of content and technical support to the nanotechnology community, including working with the community in curating specialized concepts and definitions, and making those concepts available in NCI Thesaurus (NCIt) and the NanoParticle Ontology (NPO) hosted on EVS systems and integrated into NCI Metathesaurus (NCIm); a
<a href="https://ncisvn.nci.nih.gov/svn/docs/trunk/calab/caNanoLabCDEandTerminologyDevelopment/caNanoLab_Glossary.pdf" rel="nofollow">
  glossary
</a>
of nanotechnology terms also is provided within the caNanoLab application (see below).
<h4 id="UserProfilesBroaderCommunity-NanotechnologyWorkingGroup">
  Nanotechnology Working Group
</h4>
The NCIP
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=8783902" rel="nofollow">
  Nanotechnology Working Group
</a>
was started as part of the caBIG® Integrative Cancer Research Workspace (ICR Nano WG), with participation of approximately 40 agencies, universities and institutes working to federate nanotechnology databases. One requirement has been to develop data and vocabulary standards to facilitate federation and increase data accessibility. EVS has been an active participant in the working group. Part of the working group effort is the continuing development of the NanoParticle Ontology (NPO), from Washington University in St. Louis, and the development of the ISA-TAB-Nano data sharing format. EVS has supported this effort in several areas: giving feedback on ontology structure and terminology best practices, supporting NPO curation using the NCI Protégé platform, loading and hosting NPO on LexEVS, and preparing NPO for integration into the NCI Metathesaurus (NCIm).
<h4 id="UserProfilesBroaderCommunity-caNanoLab">
  caNanoLab
</h4>
The
<a href="http://cssi.cancer.gov/nano.asp" rel="nofollow">
  NCI Office of Cancer Nanotechnology Research (OCNR)
</a>
partnered with CBIIT and the
<a href="http://ncl.cancer.gov/" rel="nofollow">
  NCI Nanotechnology Characterization Laboratory (NCL)
</a>
in 2006 to develop a data sharing platform called
<a href="https://cananolab.nci.nih.gov/" rel="nofollow">
  caNanoLab
</a>
. caNanoLab has a goal of semantic interoperability across centers performing nanoparticle characterization studies. caNanoLab is based on an information model representing nanoparticles and their physical and in vitro characterization. NCI Thesaurus (NCIt) has been supporting concept curation for development of caNanoLab since its inception. NCIt editors have worked with the developers and users of caNanoLab from the CCNEs and other academic institutions to define concepts for data curation and to expand the object model to include data submission for both characterizations of experimental nanomaterials and translational research studies.The infrastructure available for further development of caNanoLab has been decreased over the last several years, but caNanoLab continues to be maintained and is heavily used. As of February 21, 2014, it contains 1,027 samples, 95 sample sources, 4,025 characterizations, and 46 protocols, and identifies 1,894 publications, while the caNanoLab home page shows 420,937 visitors since June 3, 2010. For more information about caNanoLab, visit the
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=25134359" rel="nofollow">
  wiki home page
</a>
.
<h4 id="UserProfilesBroaderCommunity-ISA-TAB-Nano">
  ISA-TAB-Nano
</h4>
ISA-TAB-Nano is being developed by a subset of participants of the NCIP Nanotechnology Working Group, which includes members representing CBIIT, Oregon State University, PNNL, Washington University St. Louis, Stanford, Jackson Labs, Pennsylvania BioNano Systems, NIOSH, NCI Frederick NCL, and Emory/Georgia Tech. ISA-TAB-Nano is a data representation format that is designed to facilitate sharing research related to the in vivo and in vitro characterization of nanomaterials and any associated small molecules or biological specimens. This format is compatible with spreadsheets or tab-delimited files. ISA-TAB-Nano is based on the existing specification developed by the
<a href="http://www.isa-tools.org" rel="nofollow">
  ISA community
</a>
, the investigation/study/assay (ISA-TAB) format specification. ISA-TAB was designed to assist in recording and sharing of both data and metadata associated with the large volume of data generated by the numerous assays and technology types used in the “omics” communities.The ISA-TAB file structure relies on three primary files: investigation, study, and assay (ISA) files. The ISA-TAB-Nano specification adds additional fields to the ISA-TAB files and an additional material file to record nanomaterial and small molecule structural and functional characteristics. ISA-TAB-Nano has been reviewed as an ASTM standard (
<a href="http://www.astm.org/SOCIETY_REVIEW/E56%20%2812-04%29.htm" rel="nofollow">
  WK28974
</a>
). Further development for this data sharing format is focused on training users, improving the usability of the format and increasing the compatibility between ISA-TAB-Nano and the ISA community curation and validation tool sets. See the
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42336563" rel="nofollow">
  ISA-TAB-Nano Specification
</a>
for more information.
<h4 id="UserProfilesBroaderCommunity-NanoParticleOntology(NPO)">
  NanoParticle Ontology (NPO)
</h4>
The NanoParticle Ontology (
<a href="http://www.nano-ontology.org/" rel="nofollow">
  NPO
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
) is loaded in NCI Metathesaurus, on LexEVS, and on the
<a href="http://nciterms.nci.nih.gov/ncitbrowser/pages/vocabulary.jsf?dictionary=NPO" rel="nofollow">
  NCI Term Browser
</a>
. NPO curators use the NCI Protégé curation tool and they use NCI Thesaurus as a cross-reference and a source for some of their definitions.EVS Related References
<ol>
  <li>
    Bailey LO, Kennedy CH, Fritts MJ, Hartel FW. Development of a model for the representation of nanotechnology-specific terminology.  AMIA Annu Symp Proc. 2006:849. PubMed PMID: 17238469; PubMed Central PMCID: PMC1839578. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/17238469" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    de la Iglesia D, Cachau RE, García-Remesal M, Maojo V. Nanoinformatics knowledge infrastructures: bringing efficient information management to nanomedical research.  Comput Sci Discov. 2013 Nov 27;6(1):014011. PubMed PMID: 24932210; PubMed Central PMCID: PMC4053539. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/24932210" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    de la Iglesia D, García-Remesal M, Anguita A, Muñoz-Mármol M, Kulikowski C, Maojo V. A Machine Learning Approach to Identify Clinical Trials Involving Nanodrugs and Nanodevices from ClinicalTrials.gov.  PLoS One. 2014 Oct 27;9(10):e110331. doi: 10.1371/journal.pone.0110331. eCollection 2014. PubMed PMID: 25347075; PubMed Central PMCID: PMC4210133. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/25347075" rel="nofollow">
      PubMed
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Klaessig FG. Developing official practices for nanoEHS data compilation, curation and compliance.  Innovation and responsibility: engaging with new and emerging technologies, S.NET 005. Coenen C, Dijkstra A, Fautz C, Guivant J, Konrad K, Milburn C, van Lente H, eds. Heidelberg: IOS Press and AKA; 2014. pp. 121–133. [
    <a href="http://www.thesnet.net/wp-content/uploads/13developing_aka_s.net005.pdf" rel="nofollow">
      PDF
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
    ]
  </li>
  <li>
    Thomas DG, Gaheen S, Harper SL, Fritts M, Klaessig F, Hahn-Dantona E, Paik D, Pan S, Stafford GA, Freund ET, Klemm JD, Baker NA. ISA-TAB-Nano: a specification for sharing nanomaterial research data in spreadsheet-based format.  BMC Biotechnol. 2013 Jan 14;13:2. doi: 10.1186/1472-6750-13-2. PubMed PMID: 23311978; PubMed Central PMCID: PMC3598649. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/23311978" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Thomas DG, Klaessig F, Harper SL, Fritts M, Hoover MD, Gaheen S, Stokes TH, Reznik-Zellen R, Freund ET, Klemm JD, Paik DS, Baker NA. Informatics and standards for nanomedicine technology.  Wiley Interdiscip Rev Nanomed Nanobiotechnol. 2011 Jun 30. doi: 10.1002/wnan.152. Epub ahead of print. PubMed PMID: 21721140. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/21721140" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Thomas DG, Pappu RV, Baker NA. NanoParticle Ontology for cancer nanotechnology research.  J Biomed Inform. 2011 Feb;44(1):59-74. Epub 2010 Mar 6. PubMed PMID: 20211274; PubMed Central PMCID: PMC3042056. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/20211274" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Zhu Z. Flash Nanoprecipitation: Prediction and Enhancement of Particle Stability via Drug Structure.  Mol Pharm. 2014 Feb 3. [Epub ahead of print] PubMed PMID: 24484077. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/24484077" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-OpenBiomedicalOntologies(OBO)">
  Open Biomedical Ontologies (OBO)
</h3>
EVS has worked with the OBO Foundry group since around 2005 to develop shared principles for open ontologies. NCI Thesaurus is designated as an Application ontology, since it uses and references domain ontologies within the OBO Foundry group. EVS makes several of the OBO Foundry ontologies available through LexEVS for community use. (For more information, visit the
<a href="http://www.obofoundry.org/" rel="nofollow">
  OBO Foundry website
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
.)EVS Related References
<ol>
  <li>
    De Coronado S, Tuttle MS, Solbrig HR. Using the UMLS Semantic Network to Validate NCI Thesaurus Structure and Analyze its Alignment with the OBO Relations Ontology.  AMIA Annu Symp Proc. 2007:165-70. [
    <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2655888/pdf/amia-0165-s2007.pdf" rel="nofollow">
      NCBI
    </a>
    ]
  </li>
</ol>
<h3 id="UserProfilesBroaderCommunity-PhenX">
  PhenX
</h3>
<a href="https://www.phenx.org/" rel="nofollow">
  PhenX
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
was initiated by National Human Genome Research Institute (NHGRI) in 2007 to develop consensus measures for phenotypes and exposures in support of genome-wide association studies (GWAS) and other large-scale research efforts. Run by RTI International with broad participation among NIH institutes and the research community, PhenX initially prioritized 21 research domains relevant to genomics research and public health. EVS provides ongoing terminology support for these domains and related PhenX efforts.
<h3 id="UserProfilesBroaderCommunity-TranslationalResearchandPatientSafetyinEurope(TRANSFoRm)">
  Translational Research and Patient Safety in Europe (TRANSFoRm)
</h3>
The
<a href="http://transformproject.eu/" rel="nofollow">
  TRANSFoRm
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
2010-2015 European Commission project for European health care reform involved a consortium of 15 European Universities and two private partners, led by King’s College London, to develop methods, standards and systems for the integration of healthcare computer systems for clinical care and research. EVS and the VKC supported TRANSFoRm terminology efforts. The TRANSFoRm Integrated Vocabulary Service (TRANSFoRm VS) used the LexEVS terminology server and other EVS resources; the VKC worked with TRANSFoRm to develop a new loader for the WHO Anatomical Therapeutic Chemical (ATC) terminology, including requirements gathering, coding, testing, and documentation; and efforts to integrate with and extend the NIH-funded electronic Primary Care Research Network (ePCRN) and ePCRN Workbench have used both LexEVS and the NCI Metathesaurus.EVS Related References
<ol>
  <li>
    Delaney BC, Curcin V, Andreasson A, Arvanitis TN, Bastiaens H, Corrigan D, Ethier JF, Kostopoulou O, Kuchinke W, McGilchrist M, van Royen P, Wagner P. Translational Medicine and Patient Safety in Europe: TRANSFoRm--Architecture for the Learning Health System in Europe.  Biomed Res Int. 2015;2015:961526. doi: 10.1155/2015/961526. Epub 2015 Oct 11. PubMed PMID: 26539547; PubMed Central PMCID: PMC4619923. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/26539547" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Ethier JF, Curcin V, Barton A, McGilchrist MM, Bastiaens H, Andreasson A, Rossiter J, Zhao L, Arvanitis TN, Taweel A, Delaney BC, Burgun A. Clinical data integration model. Core interoperability ontology for research using primary care data.  Methods Inf Med. 2015;54(1):16-23. doi: 10.3414/ME13-02-0024. Epub 2014 Jun 18. PubMed PMID: 24954896. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/24954896" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
  <li>
    Ethier JF, Dameron O, Curcin V, McGilchrist MM, Verheij RA, Arvanitis TN, Taweel A, Delaney BC, Burgun A. A unified structural/terminological interoperability framework based on LexEVS: application to TRANSFoRm.  J Am Med Inform Assoc. 2013 Sep 1;20(5):986-94. doi: 10.1136/amiajnl-2012-001312. Epub 2013 Apr 9. [
    <a href="http://www.ncbi.nlm.nih.gov/pubmed/23571850" rel="nofollow">
      PubMed
    </a>
    ]
  </li>
</ol>
Help Downloading FilesFor help accessing PDF, audio, video, and compressed files on this wiki, go to
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=328991650" rel="nofollow">
  Help Downloading Files
</a>
.  