<h3 id="ViewingDatawithIGVandHMV-ViewingDatawiththeIntegrativeGenomicsViewer">
  Viewing Data with the Integrative Genomics Viewer
</h3>
Once you have run a query for
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=48726023" rel="nofollow">
  gene expression
</a>
, or have run analyses for
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=48726023" rel="nofollow">
  copy number
</a>
, or analyses for genomic data, you can view results in the Integrative Genomics Viewer (IGV).The IGV is a high-performance visualization tool for interactive exploration of large, integrated datasets. It supports a wide variety of data types including sequence alignments, microarrays, and genomic annotations.IGV informationFor more information about the Integrative Genomics Viewer or to connect independently to the IGV home page, see
<a href="http://www.broadinstitute.org/software/igv/home" rel="nofollow">
  Integrative Genomics Viewer
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
.You may also want to refer to the
<a href="http://www.broadinstitute.org/software/igv/UserGuide" rel="nofollow">
  IGV User Guide
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
. The IGV viewer and the NCI Heat Map viewer both require you to install a version of Java containing Java Web Start. For more information, see
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
  Java for IGV and Heat Map Viewer
</a>
.There are two ways to integrate caIntegrator with the IGV. To configure the connection to IGV, follow one of these methods.
<h6 id="ViewingDatawithIGVandHMV-Method1IGV">
  Method 1 IGV
</h6>
<ol>
  <li>
    With the appropriate study open, at the bottom of the Query Results page, click the View in Integrative Genomics Viewer button.
  </li>
  <li>
    If you click the button at the bottom of the page with any of the query results line items selected, caIntegrator creates IGV files, with a monitor informing you of this. After the files are created, click the Launch Integrative Viewer hypertext link.
  </li>
  <li>
    Follow the instructions through the intermediate dialog boxes. After clicking Open with the Java program listed, the IGV.jnlp opens, displaying the dataset in the computer screen. An example displays in the following figure.
    <img alt="illustration of step, described in text" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caIntegrator/Viewing%20Data%20with%20IGV%20and%20HMV/IGV%20session80.PNG?api=v2">
  </li>
  <li>
    Move your mouse to hover over the genes graphic at the bottom of the page, indicated in the figure.
  </li>
  <li>
    Click the mouse when you&#39;ve identified a gene of interest. This opens the genome site at UCSC, where you can learn more about the gene. The following figures exhibits the kind of metadata you can expect from the UCSC genome site.
    <img alt="illustration of step, described in text" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caIntegrator/Viewing%20Data%20with%20IGV%20and%20HMV/GABRG1%20at%20UCSC80.PNG?api=v2">
  </li>
</ol>
<h6 id="ViewingDatawithIGVandHMV-Method2IGV">
  Method 2 IGV
</h6>
<ol>
  <li>
    With the appropriate study open, click Integrative Genomics Viewer on the left sidebar. This opens the View IGV Selector page, shown in the following figure.
    <img alt="illustration of step, described in text" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caIntegrator/Viewing%20Data%20with%20IGV%20and%20HMV/IGV%20Selector80.PNG?api=v2">
  </li>
  <li>
    In the drop-down list, select the Gene Expression Platform for the data you want to view.
  </li>
  <li>
    Select the Copy Number Platform ID.
  </li>
  <li>
    The Annotations - Default panel displays existing annotation fields for the gene expression data in the open study. Select those fields you want to view when you open the IGV. Use the buttons for convenience if you want to Select All or Unselect All, when all are checked.
  </li>
  <li>
    Click View to see the data in the Integrative Genomic Viewer. caIntegrator creates IGV files of the data.
  </li>
  <li>
    After the files are created, click the Launch Integrative Viewer hypertext link that appears.
  </li>
  <li>
    Continue with Step 3 in
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
      Method 1 IGV
    </a>
    .
  </li>
</ol>
<h3 id="ViewingDatawithIGVandHMV-ViewingDatawithHeatMapViewer">
  Viewing Data with Heat Map Viewer
</h3>
Once you have run a query for
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=48726023" rel="nofollow">
  gene expression
</a>
, or for
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=48726023" rel="nofollow">
  copy number
</a>
, or have run analyses on genomic data, you can view results in the Heat Map Viewer (HMV).HMV informationFor more information about the Heat Map Viewer or to connect independently to the HMV home page, see
<a href="https://cgwb.nci.nih.gov/cgi-bin/heatmap" rel="nofollow">
  Heat Map Viewer documentation
</a>
or
<a href="https://cgwb.nci.nih.gov/goldenPath/heatmap/documentation/index.html" rel="nofollow">
  HMV documentation
</a>
. The IGV viewer and the NCI Heat Map viewer both require you to install a version of Java containing Java Web Start. For more information, see
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
  Java for IGV and Heat Map Viewer.
</a>
.There are two ways to integrate caIntegrator with the Heat Map Viewer. To configure the connection, follow one of these methods.
<h6 id="ViewingDatawithIGVandHMV-Method1HMV">
  Method 1 HMV
</h6>
<ol>
  <li>
    With the appropriate study open, at the bottom of the Query Results page, click the View in Heat Map Viewer button.
  </li>
  <li>
    If you click the button at the bottom of the page with any of the query results line items selected, caIntegrator creates HMV files, with a monitor informing you of this. After the files are created, click the Launch Heat Map Viewer hypertext link.
  </li>
  <li>
    Follow the instructions through the intermediate dialog boxes. After clicking Open with the Java program listed, the runs, displaying the dataset in the computer screen. An example displays in the following figure.
    <img alt="illustration of step, described in text" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caIntegrator/Viewing%20Data%20with%20IGV%20and%20HMV/heat%20map%20viewer%20results280.PNG?api=v2">
  </li>
</ol>
<h6 id="ViewingDatawithIGVandHMV-Method2HMV">
  Method 2 HMV
</h6>
<ol>
  <li>
    With the appropriate study open, click Heat Map Viewer on the left sidebar. This opens the View Heat Map Viewer Selector page, shown in the following figure.
    <img alt="illustration of step, described in text" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caIntegrator/Viewing%20Data%20with%20IGV%20and%20HMV/view%20heat%20map%20selector80.PNG?api=v2">
  </li>
  <li>
    Select the appropriate Copy Number Platform in the drop down list.
  </li>
  <li>
    The Annotations - Default panel displays existing annotation fields for the gene expression data in the open study. Select one or more annotations in the annotation list. For convenience, you can use the Select All or Unselect All buttons.
  </li>
  <li>
    Click View to view the data you select in Heat Map Viewer. caIntegrator creates Heat Map Viewer files of the data.
  </li>
  <li>
    After the files are created, click the Launch Heat Map Viewer hypertext link that appears.
  </li>
  <li>
    Continue with Step 3 in
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
      Method 1 HMV
    </a>
    .HMV help filesFor interpretation of the results and using HMV features, see the help files opened from HMV.
  </li>
</ol>
<h4 id="ViewingDatawithIGVandHMV-JavaforIGVandHeatMapViewer">
  Java for IGV and Heat Map Viewer
</h4>
To use the IGV and the NCI Heat Map viewer, described in
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
  Viewing Data with the Integrative Genomics Viewer
</a>
and
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=50694583" rel="nofollow">
  Viewing Data with Heat Map Viewer
</a>
, you must install a version of Java containing Java Web Start. You must install recent versions of the Java Development Kit (JDK 1.5.0 aka JDK 5.0 or newer) or Java Runtime Environment (JRE 1.5.0 aka JRE 5.0 or newer). The easiest option is to install
<a href="http://www.java.com/en/download/faq/java_webstart.xml" rel="nofollow">
  JRE 5.0
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
Without Java Web Start, when you click Launch Integrative Genomics Viewer or Launch Heat Map Viewer, a dialog box displays in your browser giving you the option to save or open with igv.jnlp (IGV) or retrieveFile.jnlp (HMV). Clicking the Open option starts the Java Web Start Launcher (default), installing the Java app so that you can view the files.Upon first launchThe first time you launch the IGV or HMV with Java properly installed, regardless of browser type, a warning may appear: the &#34;the digital signature cannot be verified&#34;. Click Run to proceed with opening the viewer.