Page Contents
<ul>
  <li>
    <a href="#Data,Metadata,andAnnotations-Data,Metadata,andAnnotationsOverview" rel="nofollow">
      Data, Metadata, and Annotations Overview
    </a>
  </li>
  <li>
    <a href="#Data,Metadata,andAnnotations-EvidenceCodes" rel="nofollow">
      Evidence Codes
    </a>
  </li>
  <li>
    <a href="#Data,Metadata,andAnnotations-RoleCodeandRoleDetailSimilarities" rel="nofollow">
      Role Code and Role Detail Similarities
    </a>
    <ul>
      <li>
        <a href="#Data,Metadata,andAnnotations-RoleCodes" rel="nofollow">
          Role Codes
        </a>
      </li>
      <li>
        <a href="#Data,Metadata,andAnnotations-RoleDetails" rel="nofollow">
          Role Details
        </a>
      </li>
    </ul>
  </li>
  <li>
    <a href="#Data,Metadata,andAnnotations-CellLineandNegationIndicators" rel="nofollow">
      Cell Line and Negation Indicators
    </a>
  </li>
  <li>
    <a href="#Data,Metadata,andAnnotations-StatusFlags" rel="nofollow">
      Status Flags
    </a>
    <ul>
      <li>
        <a href="#Data,Metadata,andAnnotations-GeneStatusFlags" rel="nofollow">
          Gene Status Flags
        </a>
      </li>
      <li>
        <a href="#Data,Metadata,andAnnotations-SentenceStatusFlags" rel="nofollow">
          Sentence Status Flags
        </a>
      </li>
    </ul>
  </li>
  <li>
    <a href="#Data,Metadata,andAnnotations-Comments" rel="nofollow">
      Comments
    </a>
  </li>
</ul>
Documentation Table of Contents
<ul>
  <li>
    <a href="https://wiki.nci.nih.gov/x/hC5yAQ" rel="nofollow">
      Documentation Main Page
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/DS9yAQ" rel="nofollow">
      Creation of the Cancer Gene Index
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/zC1yAQ" rel="nofollow">
      Data, Metadata, and Annotations
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
      Cancer Gene Index Gene-Disease and Gene-Compound XML Documents
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/7zByAQ" rel="nofollow">
      caBIO APIs
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/hzJyAQ" rel="nofollow">
      Cancer Gene Index Shared Parsed Data and Code
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/0C9yAQ" rel="nofollow">
      caBIO Portlet Templated Searches
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/TTByAQ" rel="nofollow">
      caBIO Home Page
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/VDFyAQ" rel="nofollow">
      caBIO iPhone Application
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/-TByAQ" rel="nofollow">
      caBIO Portlet Simple Searches
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/si9yAQ" rel="nofollow">
      Glossary
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/yS9yAQ" rel="nofollow">
      Credits and Resources
    </a>
  </li>
</ul>
To Print the GuideWe recommend you print one wiki page of the guide at a time. To do this, click the printer icon at the top right of the page; then from the browser File menu, choose Print. Printing multiple pages at one time is more complex. For instructions, refer to
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=8168758" rel="nofollow">
  Printing multiple pages
</a>
 .Having Trouble Reading the Text?Resizing the text for any web page is easy. For information on how to do this in your web browser, refer to this
<a href="http://www.w3.org/WAI/changedesign" rel="nofollow">
  W3C tutorial
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
<h1 id="Data,Metadata,andAnnotations-Data,Metadata,andAnnotationsOverview">
  Data, Metadata, and Annotations Overview
</h1>
All of the Gene-Disease and Gene-Compound Cancer Gene Index data, annotations, and metadata are available in the two XML documents and the caBIO interfaces, excluding the caBIO Portlet Templated Search. These data include NCI Thesaurus disease and compound terms, NCI Thesaurus disease and compound concept identifiers, HUGO Gene Symbols, LocusLink Gene Symbols, UniProt Identifiers, the sentences that contained evidence of the gene-disease or gene-compound associations, and the PubMed identifier of the abstract from which each sentence was extracted. The gene-disease and gene-compound data were then annotated by human curators with Evidence Codes, Role Codes, and Role Details; Cell line and Negation Indicators; Sentence and Gene Status Flags; and Comments.For additional information on how these data were collected, validated, and annotated, refer to the
<a href="https://wiki.nci.nih.gov/x/DS9yAQ" rel="nofollow">
  Creation of the Cancer Gene Index
</a>
section.
<h1 id="Data,Metadata,andAnnotations-EvidenceCodes">
  Evidence Codes
</h1>
Evidence codes qualify the assertions with respect to the association of a gene to a disease or compound term by telling how the assertions were made (for example, through inference or experimental data). The curators may have identified the means by which an assertion using the extracted sentence, alone, or through careful reading of the abstract from which the sentence originated. These codes follow the suggestions of
<a href="http://www.ncbi.nlm.nih.gov/pubmed/14992503?itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum&amp;ordinalpos=1" rel="nofollow">
  Karp et al.
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
for ontologies used in pathway and genome databases. The Evidence Code associated with a specific gene-disease or gene-compound pair is found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
EvidenceCode element and is the EvidenceCode attribute of the caBIO Evidence Code class, gov.nih.nci.cabio.domain.EvidenceCode.
<table>
  <tbody>
    <tr>
      <th>
        Evidence Code
      </th>
      <th>
        Description
      </th>
    </tr>
    <tr>
      <td>
        EV-IC
      </td>
      <td>
        Inferred by curator. An assertion was inferred by a curator from relevant information such as other assertions in a database.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP
      </td>
      <td>
        Inferred from computation. The evidence for an assertion comes from a computational analysis. The assertion itself might have been made by an author or by a computer, that is, EV-COMP does not specify whether manual interpretation of the computation occurred.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-HINF
      </td>
      <td>
        Human inference. A curator or author inferred this assertion after review of one or more possible types of computational evidence such as sequence similarity, recognized motifs or consensus sequence, etc. When the inference was made by a computer in an automated fashion, use EV-AINF.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-HINF-SIMILAR-TO-CONSENSUS
      </td>
      <td>
        An author inferred, or reviewed a computer inference of, sequence function based on similarity to a consensus sequence.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-HINF-POSITIONAL-IDENTIFICATION
      </td>
      <td>
        An author inferred, or reviewed a computer inference of, promoter position relative to the -10 and -35 boxes.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-HINF-FN-FROM-SEQ
      </td>
      <td>
        An author inferred, or reviewed a computer inference of, gene function based on sequence, profile, or structural similarity (as computed from sequence) to one or more other sequences.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-AINF
      </td>
      <td>
        Automated inference. A computer inferred this assertion through one of many possible methods such as sequence similarity, recognized motifs or consensus sequence, etc. When a person made the inference from computational evidence, use EV-HINF.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-AINF-SINGLE-DIRECTON
      </td>
      <td>
        Automated inference of transcription unit based on single-gene direction. Existence of a single-gene transcription unit for gene G is inferred computationally by the existence of upstream and downstream genes transcribed in the opposite direction of G.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-AINF-SIMILAR-TO-CONSENSUS
      </td>
      <td>
        A DNA sequence similar to previously known consensus sequences is computationally identified.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-AINF-POSITIONAL-IDENTIFICATION
      </td>
      <td>
        Automated inference of promoter position relative to the -10 and -35 boxes.
      </td>
    </tr>
    <tr>
      <td>
        EV-COMP-AINF-FN-FROM-SEQ
      </td>
      <td>
        Automated inference of function from sequence. A computer inferred a gene function based on sequence, profile, or structural similarity (as computed from sequence) to one or more other sequences.
      </td>
    </tr>
    <tr>
      <td>
        EV-AS-TAS
      </td>
      <td>
        Traceable author statement. The assertion was made in a publication – such as a review – that itself did not describe an experiment supporting the assertion. The statement referenced another publication that supported the assertion, but it is unclear whether that publication described an experiment that supported the assertion.
      </td>
    </tr>
    <tr>
      <td>
        EV-AS-NAS
      </td>
      <td>
        Non-traceable author statement. The assertion was made in a publication such as a review, without a reference to a publication describing an experiment that supports the assertion.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP
      </td>
      <td>
        Inferred from experiment. The evidence for an assertion comes from a wet-lab experiment of some type.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IPI
      </td>
      <td>
        IPI inferred from physical interaction The assertion was inferred from a physical interaction such as 2-hybrid interactions, Co-purification, Co-immunoprecipitation, Ion/protein binding experiments This code covers physical interactions between the gene product of interest and another molecule (or ion, or complex). For functions such as protein binding or nucleic acid binding, a binding assay is simultaneously IPI and IDA; IDA is preferred because the assay directly detects the binding.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA
      </td>
      <td>
        IDA inferred from direct assay. The assertion was inferred from a direct experimental assay such as Enzyme assays, In vitro reconstitution (for example, transcription), Immunofluorescence, Cell fractionation, etc.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-UNPURIFIED-PROTEIN
      </td>
      <td>
        Direct assay of unpurified protein. Presence of a protein activity is indicated by an assay. However, the precise identity of the protein with that activity is not established by this experiment (protein has not been purified).
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-TRANSCRIPTION-INIT-MAPPING
      </td>
      <td>
        The transcription start site is identified by primer extension.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-TRANSCRIPT-LEN-DETERMINATION
      </td>
      <td>
        The length of the (transcribed) RNA is experimentally determined. The length of the mRNA is compared with that of the DNA sequence and by this means the number of genes transcribed are established.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-RNA-POLYMERASE-FOOTPRINTING
      </td>
      <td>
        The binding of RNA polymerase to a DNA region (the promoter) is shown by footprinting.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-PURIFIED-PROTEIN-MULTSPECIES
      </td>
      <td>
        Protein purified from mixed culture or other multispecies environment (such as, infected plant or animal tissue), and activity measured through in vitro assay.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-PURIFIED-PROTEIN
      </td>
      <td>
        Protein purified to homogeneity from specific species (or from heterologous expression vector), and activity measured through in vitro assay.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-BOUNDARIES-DEFINED
      </td>
      <td>
        Sites or genes bounding the transcription unit are experimentally identified. Several possible cases exist, such as defining the boundaries of a transcription unit with an experimentally identified promoter and terminator, or with a promoter and a downstream gene that is transcribed in the opposite direction, or with a terminator and an upstream gene that is transcribed in the opposite direction.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-BINDING-OF-PURIFIED-PROTEINS
      </td>
      <td>
        IDA inferred from direct assay. The assertion was inferred from a direct experimental assay such as Enzyme assays, In vitro reconstitution (for example, transcription), Immunofluorescence, Cell fractionation.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IDA-BINDING-OF-CELLULAR-EXTRACTS
      </td>
      <td>
        There exists physical evidence of the binding of cellular extracts containing a regulatory protein to its DNA binding site. This can be either by footprinting or mobility shift assays.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IEP
      </td>
      <td>
        IEP inferred from expression pattern. The assertion was inferred from a pattern of expression data such as Transcript levels (for example, Northerns, microarray data), Protein levels (for example, Western blots).
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IEP-GENE-EXPRESSION-ANALYSIS
      </td>
      <td>
        The expression of the gene is analyzed through a transcriptional fusion (that is, lacZ), and a difference in expression levels is observed when the regulatory protein is present (wild type) vs in its absence. Note that this evidence does not eliminate the possibility of an indirect effect of the regulator on the regulated gene.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IGI
      </td>
      <td>
        IGI inferred from genetic interaction. The assertion was inferred from a genetic interaction such as &#34;Traditional&#34; genetic interactions such as suppressors, synthetic lethals, etc., Functional complementation, Inference about one gene drawn from the phenotype of a mutation in a different gene. This category includes any combination of alterations in the sequence (mutation) or expression of more than one gene/gene product. This category can therefore cover any of the IMP experiments that are done in a non-wild-type background, although we prefer to use it only when all mutations are documented.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IGI-FUNC-COMPLEMENTATION
      </td>
      <td>
        Protein activity inferred by isolating its gene and performing functional complementation of a well characterized heterologous mutant for the protein.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IMP
      </td>
      <td>
        IMP inferred from mutant phenotype. The assertion was inferred from a mutant phenotype such as Any gene mutation/knockout, Overexpression/ectopic expression of wild-type or mutant genes, Anti-sense experiments, RNA interference experiments, Specific protein inhibitors, Complementation. Inferences made from examining mutations or abnormal levels of only the product(s) of the gene of interest are covered by code EV-IMP (compare to code EV-IGI). Use this code for experiments that use antibodies or other specific inhibitors of RNA or protein activity, even though no gene may be mutated (the rationale is that EV-IMP is used where an abnormal situation prevails in a cell or organism).
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IMP-REACTION-ENHANCED
      </td>
      <td>
        Gene is isolated and over-expressed, and increased accumulation of reaction product is observed.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IMP-POLAR-MUTATION
      </td>
      <td>
        If a mutation in a gene or promoter prevents expression of the downstream genes due to a polar effect, the mutated gene is clearly part of the transcription unit.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IMP-REACTION-BLOCKED
      </td>
      <td>
        Mutant is characterized, and blocking of reaction is demonstrated.
      </td>
    </tr>
    <tr>
      <td>
        EV-EXP-IMP-SITE-MUTATION
      </td>
      <td>
        A cis-mutation in the DNA sequence of the transcription-factor binding site interferes with the operation of the regulatory function. This is considered strong evidence for the existence and functional role of the DNA binding site.
      </td>
    </tr>
    <tr>
      <td>
        not_assigned
      </td>
      <td>
        Evidence Code was not assigned.
      </td>
    </tr>
    <tr>
      <td>
        based on abstract
      </td>
      <td>
        Determinations of whether the gene-disease or gene-compound association from a sentence was factual based upon an expert&#39;s interpretation of the abstract from which the sentence originated.
      </td>
    </tr>
    <tr>
      <td></td>
      <td>
        No Evidence Code was assigned because the sentence did not contain the expected gene-disease or gene-compound association evidence.
      </td>
    </tr>
  </tbody>
</table>
<h1 id="Data,Metadata,andAnnotations-RoleCodeandRoleDetailSimilarities">
  Role Code and Role Detail Similarities
</h1>
The Cancer Gene Index Role Codes and Role Details are derived from NCI Role Codes. Both describe the semantic associations between gene concept and either a disease or compound concept (that is, concept pairs). Whereas the Evidence Codes describe how the association was inferred or the type of experiment upon which the inference was made, Role Codes and Role Details give information about the actual gene-disease or gene-compound association. Multiple Role Codes and Role Details can be used for the same sentence.NoteA concept is the actual compound, disease, or gene to which the various names, acronyms, alternate spellings, and abbreviations refer.
<h2 id="Data,Metadata,andAnnotations-RoleCodes">
  Role Codes
</h2>
Gene-Disease and Gene-Compound Role Codes most often describe that a gene is associated with a disease or compound (for example, GENE_ASSOCIATED_WITH_DISEASE) or how the concepts are associated (for example, Chemical_or_Drug_Is_Metabolized_By_Enzyme), but they also may describe relevant features of the role of a particular gene (for example, GENE_HAS_FUNCTION). For the former, the gene name, Role Code, and disease or compound often can form a sentence, such as &#34;BRCA1 GENE_ASSOCIATED_WITH_DISEASE BREAST CANCER.&#34; The Role Code not_assigned indicates that the curator did not or could not assign a specific code. The Role Code associated describing evidence of a gene-disease or gene-compound pair is found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
PrimaryNCIRoleCode element and is the role attribute of the caBIO GeneDiseaseAssociation and GeneAgentAssociation classes, gov.nih.nci.cabio.domain.GeneDiseaseAssociation and gov.nih.nci.cabio.domain.GeneAgentAssociation.NoteAlthough pharmacological substances are referred to as &#34;compounds&#34; in the Cancer Gene Index, the NCI Thesaurus and caBIO use the term &#34;agent.&#34;
<ul>
  <li>
    Gene-Disease Role Codes
    <ul>
      <li>
        Gene_Associated_With_Disease
      </li>
      <li>
        Gene_Product_Anormaly_Affects_Pathway
      </li>
      <li>
        Gene_Product_Anomaly_Related_To_Gene_Anormaly
      </li>
      <li>
        Gene_Product_Encoded_By_Gene
      </li>
      <li>
        Gene_Product_Expressed_In_Tissue
      </li>
      <li>
        Gene_Product_Has_Associated_Anatomie
      </li>
      <li>
        Gene_Product_Has_Biochemical_Function
      </li>
      <li>
        Gene_Product_Has_Chemical_Classification
      </li>
      <li>
        Gene_Product_Has_Malfunction_Type
      </li>
      <li>
        Gene_Product_Has_Organism_Source
      </li>
      <li>
        Gene_Product_Has_Structural_Domain_Or_Motif
      </li>
      <li>
        Gene_Product_is_Biomarker_of
      </li>
      <li>
        Gene_Product_is_Biomarker_Type
      </li>
      <li>
        Gene_Product_is_Pathway_Element
      </li>
      <li>
        Gene_Product_is_Physical_Part_Of
      </li>
      <li>
        Gene_Product_Malfunction_Associated_With_Disease
      </li>
      <li>
        Gene_Product_Plays_Role_In_Biological_Process
      </li>
      <li>
        Gene_Malfunction_Associated_With_Disease
      </li>
      <li>
        Gene_Expressed_In_Tissue
      </li>
      <li>
        Gene_Found_In_Organism
      </li>
      <li>
        Gene_Has_Anormally
      </li>
      <li>
        Gene_Has_Clone
      </li>
      <li>
        Gene_Has_Expression_Measurement
      </li>
      <li>
        Gene_Has_Function
      </li>
      <li>
        Gene_In_Chromosomal_Location
      </li>
      <li>
        Gene_is_Biomarker_of
      </li>
      <li>
        Gene_Is_Pathway_Element
      </li>
      <li>
        Gene_Plays_Role_In_Process
      </li>
      <li>
        Disease_Has_Cytogenetic_Abnormality
      </li>
      <li>
        Disease_May_Have_Cytogenetic_Abnormality
      </li>
      <li>
        Disease_Has_Molecular_Abnormality
      </li>
      <li>
        Disease_May_Have_Molecular_Abnormality
      </li>
    </ul>
  </li>
  <li>
    Gene-Compound Role Codes
    <ul>
      <li>
        Chemical_or_Drug_Affects_Cell_Type_or_Tissue
      </li>
      <li>
        Chemical_or_Drug_Plays_Role_in_Biological_Process
      </li>
      <li>
        Chemical_or_Drug_FDA_Approved_for_Disease
      </li>
      <li>
        Chemical_or_Drug_Is_Metabolized_By_Enzyme
      </li>
      <li>
        Chemical_or_Drug_Has_Accepted_Therapeutic_Use_For
      </li>
      <li>
        Chemical_or_Drug_Has_Study_Therapeutic_Use_For
      </li>
      <li>
        Chemical_or_Drug_Has_Mechanism_Of_Action
      </li>
      <li>
        Chemical_or_Drug_Affects_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_Has_Target_Gene_Product
      </li>
    </ul>
  </li>
</ul>
<h2 id="Data,Metadata,andAnnotations-RoleDetails">
  Role Details
</h2>
Gene-Disease and Gene-Compound Role Details most often provide precise descriptions of the association of a gene term and a corresponding disease or compound term. These Details can also describe relevant features of the role of a particular gene (for example, Chemical_or_Drug_Represses_Gene_Product_Expression). While similar to Role Codes, Role Details give more specific semantic descriptions. For example, a Role Detail for a particular gene-disease concept pair association may be GENE_PRODUCT_UPREGULATED_IN_DISEASE, whereas a similar role code may be GENE_ASSOCIATED_WITH_DISEASE. The Role Detail not_assigned indicates that the curator did not or could not assign a specific semantic detail. The Role Detail associated with a specific gene-disease or gene-compound pair is found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
OtherRole element and is, like Role Codes, the role attribute of the caBIO GeneDiseaseAssociation and GeneAgentAssociation classes.
<ul>
  <li>
    Gene-Disease Role Details
    <ul>
      <li>
        Gene_Product_Affects_Disease
      </li>
      <li>
        Gene_Product_Affects_Disease_Process
      </li>
      <li>
        Gene_Product_Expressed_in_Disease
      </li>
      <li>
        Gene_Product_Decreased_in_Disease
      </li>
      <li>
        Gene_Product_Increased_in_Disease
      </li>
      <li>
        Gene_Product_Level_Changed_in_Disease
      </li>
      <li>
        Gene_Expressed_in_Disease
      </li>
      <li>
        Gene_Expression_Downregulated_in_Disease
      </li>
      <li>
        Gene_Expression_Upregulated_in_Disease
      </li>
      <li>
        Gene_Expression_Changed_in_Disease
      </li>
      <li>
        Gene_May_Be_Associated_With_Disease
      </li>
      <li>
        Gene_Anormaly_has_Disease-Related_Function
      </li>
      <li>
        Gene_Anormaly_May_have_Disease-Related_Function
      </li>
      <li>
        Gene_Product_Anormaly_has_Disease-Related_Function
      </li>
      <li>
        Gene_Product_Anormaly_May_have_Disease-Related_Function
      </li>
      <li>
        Gene_has_Therapeutic_Relevance
      </li>
      <li>
        Gene_May_have_Therapeutic_Relevance
      </li>
      <li>
        Gene_Product_has_Therapeutic_Relevance
      </li>
      <li>
        Gene_Product_May_have_Therapeutic_Relevance
      </li>
    </ul>
  </li>
  <li>
    Gene-Compound Role Details
    <ul>
      <li>
        Chemical_or_Drug_in_Clinical_Study
      </li>
      <li>
        Chemical_or_Drug_May_Affect_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_May_Affect_Gene
      </li>
      <li>
        Chemical_or_Drug_Affects_Gene
      </li>
      <li>
        Chemical_or_Drug_Regulates_Gene
      </li>
      <li>
        Chemical_or_Drug_Regulates_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_Activates_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_Inhibits_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_Affects_Gene_Product_Function
      </li>
      <li>
        Chemical_or_Drug_Binds_to_Gene_Product
      </li>
      <li>
        Chemical_or_Drug_Affects_Expression
      </li>
      <li>
        Chemical_or_Drug_Affects_Gene_Expression
      </li>
      <li>
        Chemical_or_Drug_Affects_Gene_Product_Expression
      </li>
      <li>
        Chemical_or_Drug_Changes_Expression
      </li>
      <li>
        Chemical_or_Drug_Induces_Gene_Expression
      </li>
      <li>
        Chemical_or_Drug_Induces_Gene_Product_Expression
      </li>
      <li>
        Chemical_or_Drug_Regulates_Expression
      </li>
      <li>
        Chemical_or_Drug_Represses_Gene_Expression
      </li>
      <li>
        Chemical_or_Drug_Represses_Gene_Product_Expression
      </li>
      <li>
        Chemical_or_Drug_Mediates_Pathway_Activity
      </li>
      <li>
        Chemical_or_Drug_Increases_Pathway_Activity
      </li>
      <li>
        Chemical_or_Drug_Decreases_Pathway_Activity
      </li>
      <li>
        Chemical_or_Drug_Mediates_Metabolic_Status
      </li>
      <li>
        Chemical_or_Drug_Increases_Metabolic_Status
      </li>
      <li>
        Chemical_or_Drug_Decreases_Metabolic_Status
      </li>
      <li>
        Chemical_or_Drug_Has_Physiologic_Effect
      </li>
      <li>
        Gene_Product_Affects_Compound
      </li>
      <li>
        Gene_Product_May_Affect_Compound
      </li>
      <li>
        Gene_Affects_Compound
      </li>
      <li>
        Gene_May_Affect_Compound
      </li>
      <li>
        Gene_Product_Antagonizes_Chemical_or_Drug
      </li>
      <li>
        Gene_Product_Transports_Compound
      </li>
      <li>
        Gene_Anomaly_Effects_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_Product_Anomaly_Effects_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_Anomaly_May_Effect_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_Product_Anomaly_May_Effect_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_is_Associated_with_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_Product_is_Associated_with_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_May_be_Associated_with_Resistance_to_Chemical_or_Drug
      </li>
      <li>
        Gene_Product_May_be_Associated_with_Resistance_to_Chemical_or_Drug
      </li>
    </ul>
  </li>
</ul>
<h1 id="Data,Metadata,andAnnotations-CellLineandNegationIndicators">
  Cell Line and Negation Indicators
</h1>
The binary indicators yes or no were set for cell line and negation annotations of each gene-disease and gene-compound association. Cell line indicators denote whether the evidence came from a cell line (yes) or other source, such a human subject, animal model, or primary cells (no). The cell line indicator is the text contents of the XML CelllineIndicator element and the celllineStatus attribute of the caBIO Evidence class. Negation indicators specify whether the evidence actually described a lack of association between the candidate binary concept pair (yes), or whether there was a true relationship between them (no). The curators may have deduced the negation indicator by the extracted sentence, alone, or through careful reading of the abstract from which the sentence originated. Occasionally, the curators did not set a negation indicator (-). The negation indicator is the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
NegationIndicator element and the negationStatus attribute of the caBIO Evidence class.
<h1 id="Data,Metadata,andAnnotations-StatusFlags">
  Status Flags
</h1>
The curators set flags to denote the state of annotations of a gene term or of a particular sentence.
<h2 id="Data,Metadata,andAnnotations-GeneStatusFlags">
  Gene Status Flags
</h2>
Gene Status Flags describe whether annotations for all of the sentences for a given gene term are complete and whether the gene term has been withdrawn from EntrezGene. All low frequency sentence count genes were finished, whereas some
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=24260365" rel="nofollow">
  high frequency sentence count genes were not
</a>
. The status of a specific gene is found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
GeneStatusFlag element.
<table>
  <tbody>
    <tr>
      <th>
        Gene Status Flag
      </th>
      <th>
        Status Flag Description
      </th>
    </tr>
    <tr>
      <td>
        Finished
      </td>
      <td>
        All sentences with this gene have been annotated
      </td>
    </tr>
    <tr>
      <td>
        New
      </td>
      <td>
        Not all sentences with this high frequency sentence count gene have been annotated.
      </td>
    </tr>
    <tr>
      <td>
        Withdrawn
      </td>
      <td>
        Gene term has been withdrawn from EntrezGene and, where possible, the term has been mapped to a valid EntrezGene term.
      </td>
    </tr>
    <tr>
      <td>
        Entry withdrawn
      </td>
      <td>
        Gene term has been withdrawn from EntrezGene and, where possible, the term has been mapped to a valid EntrezGene term.
      </td>
    </tr>
  </tbody>
</table>
<h2 id="Data,Metadata,andAnnotations-SentenceStatusFlags">
  Sentence Status Flags
</h2>
Unlike gene status flags, which can cover many sentences associated with a single gene, Sentence Status Flags describe the curator&#39;s findings for a specific sentence. Sentences can be true positives, false positives, unclear, or redundant. The status of a specific gene is found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
SentenceStatusFlag element and is the sentenceStatus attribute of the caBIO Evidence class.
<table>
  <tbody>
    <tr>
      <th>
        Sentence Status Flag
      </th>
      <th>
        Status Flag Description
      </th>
    </tr>
    <tr>
      <td>
        Finished
      </td>
      <td>
        Sentence validation and annotation complete
      </td>
    </tr>
    <tr>
      <td>
        No_fact
      </td>
      <td>
        Invalid sentence or false positive
      </td>
    </tr>
    <tr>
      <td>
        Unclear
      </td>
      <td>
        Sentence included both a gene and disease or gene and compound term, but the relationship between the gene-disease or gene-compound pair was not obvious from the sentence.
      </td>
    </tr>
    <tr>
      <td>
        Redundant
      </td>
      <td>
        Identical gene-disease or gene-compound associations were captured from multiple sentences originating from the same abstract.
      </td>
    </tr>
  </tbody>
</table>
<h1 id="Data,Metadata,andAnnotations-Comments">
  Comments
</h1>
Often, the expert curators made free-text comments on records within the Gene-Disease or Gene-Compound databases. Comments included, but were not limited to, notations of genetic anomalies (for example, loss of heterozygosity, polymorphisms, or aberrant methylation), additional disease information, name of the non-human organism from which the experimental data were collected, information on the cell line or other notable reagents used in the execution of the experiment, and other miscellaneous information. Any comments on a sentence are found in the text contents of the
<a href="https://wiki.nci.nih.gov/x/8i1yAQ" rel="nofollow">
  XML
</a>
Comments element and the comment attribute of the caBIO Evidence class.