Data repositories are important tools in cancer research. They provide safe and sustainable locations to store data, provide access to input data for meta-analyses, and allow researchers to collaborate and share information across a common resource.The problem is that repositories are often not flexible enough to store data that do not conform to known standards. Genomics, for example, benefits from community genomics standards and open industry standards. Many other fields of study are without such standards, yet generate significant amounts of data. Lacking standards prohibits these data sets from being more accessible, easily retrieved, and potentially reused by the research community.FNLCR supports the Center for Strategic Scientific Initiatives (CSSI), an innovations center within NCI, by managing science and technology initiatives to support development programs critical to nearly all cancer and biomedical research programs in new higher risk areas, including data management, high definition single cell analysis, immuno-mass cytometry, clinical proteomics, and antibody characterization that may question existing paradigms and lead to hypothesis testing. A knowledgeable team of software developers in DSITP, led by Uma Mudunuri and managed by Andrew Quong, have developed the
<a href="https://fr-s-bsg-cssi-p.ncifcrf.gov/cssiportal/" rel="nofollow">
  CSSI DCC Portal
</a>
, a repository for CSSI DCC data and other emerging data types without a standard approach to data storage. Experimental details or metadata of the data currently in the DCC conform to the standard Investigation-Study-Assay tab-delimited format (ISA-TAB) format, which describes a scientific investigation, its study or studies, and each study&#39;s assay(s).The primary goals of the FNLCR CSSI Data Coordination Center (DCC) are to facilitate access to research data for the greater cancer research community.  Facilitating this is the DCC&#39;s design approach, which follows
<a href="https://www.force11.org/group/fairgroup/fairprinciples" rel="nofollow">
  FAIR
</a>
 (Findable, Accessible, Interoperable, and Reusable) principles for scientific data management and stewardship, applies metadata standards, and involves interactions with the research community to seek out and apply their best practices. The CSSI DCC Portal is powerful and serves the following purposes:
<ul>
  <li>
    Provides a common location and web access to data from disparate data types including gene expression results from Next Generation Sequencing, microarray experiments, histopathological images, metabolomics data, and proteomics data, allowing for easy access by multiple collaborators and researchers located at different geographic locations. It is flexible enough to handle new and unspecified data types.
  </li>
  <li>
    Provides a mechanism to identify and search for data sets stored at different locations but generated through the same project; for example, clinical and genomics data sets from a project might be deposited to a database of Genotypes and Phenotypes (dbGaP) or Genomics Data Commons (GDC), imaging files might be located at The Cancer Imaging Archives (TCIA), and cell motility information might be uploaded to the CSSI DCC.
  </li>
  <li>
    Supports storing the data in one common location so that you can make biological insights that would otherwise be missed by having data in multiple locations.
  </li>
  <li>
    Applies the information gained from one study to multiple studies and projects.
  </li>
  <li>
    Allows you to search the metadata from each study to identify datasets of interest.
  </li>
  <li>
    Develops data storage and data mining modules that can be applied across projects, avoiding duplication of effort and saving costs.
  </li>
  <li>
    Develops and/or adopts common vocabularies, data standards, and ontologies for data representation, storage, and comparison.
  </li>
</ul>
See
<a href="https://insite.cancer.gov/message/1680#1680" rel="nofollow">
   Neo4j - Rapidly Prototyping a Semantic Graph
</a>
 for more information about technologies used in CSSI DCC. The CSSI DCC Portal project team includes:
<table>
  <colgroup>
    <col/>
    <col/>
  </colgroup>
  <tbody>
    <tr>
      <th colspan="1">
        Member
      </th>
      <th colspan="1">
        Role
      </th>
    </tr>
    <tr>
      <td>
        Uma Mudunuri
      </td>
      <td>
        IT Lead
      </td>
    </tr>
    <tr>
      <td>
        Paul Donovan
      </td>
      <td>
        Team Lead/Solutions Architect
      </td>
    </tr>
    <tr>
      <td>
        David Mott
      </td>
      <td>
        Developer
      </td>
    </tr>
    <tr>
      <td>
        Mahesh Yelisetti
      </td>
      <td>
        QA Analyst
      </td>
    </tr>
    <tr>
      <td>
        Rajani Kuchipudi
      </td>
      <td>
        QA Analyst
      </td>
    </tr>
    <tr>
      <td>
        Paul Aiyetan
      </td>
      <td>
        Bioinformatician
      </td>
    </tr>
    <tr>
      <td>
        Carolyn Klinger
      </td>
      <td>
        Technical Documentation
      </td>
    </tr>
    <tr>
      <td>
        Deb Hope
      </td>
      <td>
        Subject Matter Expert
      </td>
    </tr>
    <tr>
      <td>
        Corinne Zeitler
      </td>
      <td>
        Project Manager
      </td>
    </tr>
    <tr>
      <td>
        Andrew Quong
      </td>
      <td>
        Project Director
      </td>
    </tr>
  </tbody>
</table>