The information and links on this page are no longer being updated and are provided for reference purposes only.RetiredThe NCI instance of the caArray database is now retired. For details about the retirement, see the
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=232685902" rel="nofollow">
  caArray Retirement Announcement
</a>
.Contents of this Page
<ul>
  <li>
    <a href="#caArray-ToolOverview" rel="nofollow">
      Tool Overview
    </a>
  </li>
  <li>
    <a href="#caArray-InstallationandDownloads" rel="nofollow">
      Installation and Downloads
    </a>
  </li>
  <li>
    <a href="#caArray-SupportforcaArray" rel="nofollow">
      Support for caArray
    </a>
  </li>
  <li>
    <a href="#caArray-Citations" rel="nofollow">
      Citations
    </a>
  </li>
  <li>
    <a href="#caArray-Integrationwithothertools" rel="nofollow">
      Integration with other tools
    </a>
  </li>
</ul>
At a Glance Details
<ul>
  <li>
    Version Number and Release Date: 2.5.3 May 2014
  </li>
  <li>
    Primary audience: Microarray data producers and application developers
  </li>
  <li>
    Grid Enabled? Yes
  </li>
  <li>
    Installation Level: Intermediate - technical assistance may be required, download may require supporting infrastructure or software
  </li>
  <li>
    System Requirements: Refer to the
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=17727900" rel="nofollow">
      System Requirements
    </a>
    section.
  </li>
</ul>
CBIIT Links
<ul>
  <li>
    <a href="https://datascience.cancer.gov" rel="nofollow">
      NBIA Cancer Data Science website
    </a>
  </li>
  <li>
    <a href="https://twitter.com/@NCIDataSci" rel="nofollow">
      NCI Cancer Data Science on Twitter @ NCIDataSci
    </a>
  </li>
</ul>
<h2 id="caArray-ToolOverview">
  Tool Overview
</h2>
<img alt="caArray logo" width="100" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray/Caarray.jpg?api=v2">
caArray is an open-source, web and programmatically accessible microarray data management system that supports the annotation of microarray data using
<a href="http://www.fged.org/projects/mage-tab" rel="nofollow">
  MAGE-TAB
</a>
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
and web-based forms. Data and annotations may be kept private to the owner, shared with user-defined collaboration groups, or made public.
<h2 id="caArray-InstallationandDownloads">
  Installation and Downloads
</h2>
<h3 id="caArray-Downloads">
  Downloads
</h3>
<ul>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=126059803" rel="nofollow">
      Download, Install or Upgrade caArray
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832516" rel="nofollow">
      Download Sample Data
    </a>
  </li>
</ul>
<h3 id="caArray-SystemRequirements">
  System Requirements
</h3>
While caArray is a web-based application, it does have a dependent software stack, which includes: MySQL 5.0.27 database; JBoss 5.1. (hosts the caArray grid service and the caArray application); and JEMS installer 1.0.2 GA (supports EJB 3.0 specification)
<table>
  <tbody>
    <tr>
      <th>
        Dependent Software
      </th>
      <th>
        Version
      </th>
      <th>
        URL
      </th>
    </tr>
    <tr>
      <td>
        J2SE Development Kit
      </td>
      <td>
        6.0 (J2SE 6.0)
      </td>
      <td>
        <a href="http://www.oracle.com/technetwork/java/archive-139210.html" rel="nofollow">
          J2SE 6.0
        </a>
        <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
          <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
        </a>
      </td>
    </tr>
    <tr>
      <td>
        Apache Ant
      </td>
      <td>
        1.7.0
      </td>
      <td>
        <a href="https://ncisvn.nci.nih.gov/svn/lsd/trunk/tools/apache-ant-1.7.0-bin.zip" rel="nofollow">
          apache-ant-1.7.0-bin.zip
        </a>
        <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
          <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
        </a>
      </td>
    </tr>
    <tr>
      <td>
        MySQL
      </td>
      <td>
        5.1
      </td>
      <td>
        <a href="http://downloads.mysql.com/archives.php" rel="nofollow">
          mysql 5.1
        </a>
        <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
          <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
        </a>
      </td>
    </tr>
    <tr>
      <td>
        JBoss for UPT
      </td>
      <td>
        5.1
      </td>
      <td>
        –
      </td>
    </tr>
    <tr>
      <td>
        JBOSS for Grid Service
      </td>
      <td>
        5.1.
      </td>
      <td>
        –
      </td>
    </tr>
    <tr>
      <td>
        JBOSS for caArray application
      </td>
      <td>
        5.1.
      </td>
      <td>
        –
      </td>
    </tr>
  </tbody>
</table>
<h3 id="caArray-GitHubRepositories">
  GitHub Repositories
</h3>
<ul>
  <li>
    <a href="https://github.com/NCIP/caarray" rel="nofollow">
      caArray Github Repository
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
  </li>
  <li>
    <a href="https://github.com/NCIP/caarray-docs" rel="nofollow">
      caArray docs GitHub Repository
    </a>
    <a href="http://www.cancer.gov/policies/linking" rel="nofollow">
      <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
    </a>
  </li>
</ul>
<h2 id="caArray-SupportforcaArray">
  Support for caArray
</h2>
You can contact the
<a href="mailto:NCIAppSupport@mail.nih.gov" rel="nofollow">
  NCI CBIIT Application Support
</a>
 if you have any questions or concerns.
<ul>
  <li>
    <a href="https://cabig-kc.nci.nih.gov/Molecular/forums/viewforum.php?f=6" rel="nofollow">
      End User Forum
    </a>
  </li>
  <li>
    <a href="https://cabig-kc.nci.nih.gov/Molecular/forums/viewforum.php?f=8" rel="nofollow">
      Developer Forum
    </a>
  </li>
  <li>
    <a href="http://list.nih.gov/archives/caarray_developers-l.html" rel="nofollow">
      caArray Developers Listserv
    </a>
  </li>
  <li>
    <a href="http://list.nih.gov/archives/caarray_users-l.html" rel="nofollow">
      caArray Users Listserv
    </a>
  </li>
  <li>
    <a href="https://tracker.nci.nih.gov/browse/ARRAY" rel="nofollow">
      Report a Defect or Request a Feature at caArray Jira Tracker
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/x/fpKNAg" rel="nofollow">
      Mini-how to Submit Bug reports or Feature Requests in JIRA
    </a>
    A tutorial on how to use JIRA for caArray.
  </li>
</ul>
<h2 id="caArray-Citations">
  Citations
</h2>
For a list of articles that discuss caArray, visit
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832127" rel="nofollow">
  caArray Citations
</a>
.
<h2 id="caArray-Integrationwithothertools">
  Integration with other tools
</h2>
<h3 id="caArray-IntegrationwithgeWorkbench">
  Integration with geWorkbench
</h3>
geWorkbench can query for data in an instance of caArray via a Java API. Several operations are supported. You can:
<ol>
  <li>
    Query for experiments, for example by platform or by species.
  </li>
  <li>
    Select from which hybridizations in an experiment you wish to download data.
  </li>
  <li>
    Download derived (summarized, probeset-level) data that has been parsed into the database. An example is the values loaded from Affymetrix CHP files (created through GCOS/MAS5). geWorkbench does not currently support retrieving or analyzing Affy CEL files directly.
  </li>
</ol>
The current release of geWorkbench, version 2.5.0, connects with caArray 2.5.0 and 2.5.1For more information, visit the
<a href="http://www.geworkbench.org" rel="nofollow">
  geWorkbench project page
</a>
.
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
The software can be
<a href="http://wiki.c2b2.columbia.edu/workbench/index.php/Download_and_Installation" rel="nofollow">
  downloaded
</a>
.
<h3 id="caArray-IntegrationwithcaIntegrator">
  Integration with caIntegrator
</h3>
caIntegrator can extract the genomic data from caArray hosted on either a private server can:
<ol>
  <li>
    Extract the genomic data into caIntegrator.
  </li>
  <li>
    Integrate the genomic data with the clinical data and imaging data in caIntegrator.
  </li>
  <li>
    Perform genomic data analysis with the built-in GenePattern Tool within caIntegrator.
  </li>
</ol>
The current release of caIntegrator, version 1.4, works with caArray 2.5.To install a local copy,
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=198574243" rel="nofollow">
  download a copy of the software
</a>
.
<h3 id="caArray-IntegrationwithGenePattern">
  Integration with GenePattern
</h3>
GenePattern has a module that can retrieve data from caArray using its Java API. The caArray2.3.0 Importer lets you retrieve raw data files from caArray by experiment name or id. You can specify the extension of the data file that you want to download. The default is .CEL.You can process CEL files using the GenePattern module ExpressionFileCreator. Once you have used ExpressionFileCreator, the data can be further analyzed using many different GenePattern analysis modules.The caArray 2.3.0 Importer module is compatible with caArray v2.3.x. You may
<a href="https://ncisvn.nci.nih.gov/svn/icr_portal/genepattern/trunk/genepattern/cabig_ey2_2008/caArrayImporter/caArray2.3.0Importer/caArray2.3.0Importer.zip" rel="nofollow">
  download the caArray2.3.0Importer
</a>
.The caArray 2.1.0 Importer module is compatible with caArray v2.1.x and v2.2.0. You may
<a href="https://ncisvn.nci.nih.gov/svn/icr_portal/genepattern/trunk/genepattern/cabig_ey2_2008/caArrayImporter/caArray2.1.0Importer/caArray2.1.0Importer.zip" rel="nofollow">
  download the caArray2.1.0Importer
</a>
.The GenePattern team has recently devleoped caArray2ImportViewer, a visualizer to import data from caArray into GenePattern. It is compatible with caArray 2.4.0.x. Please see the
<a href="https://ncisvn.nci.nih.gov/svn/icr_portal/genepattern/trunk/genepattern/cabig_2011/CaArray2ImportViewer/doc/CaArray2ImportViewer.pdf" rel="nofollow">
  documentation
</a>
to learn more.You may also
<a href="http://www.genepattern.org" rel="nofollow">
  download GenePattern
</a>
.
<a href="http://www.cancer.gov/policies/linking" rel="nofollow">
  <img alt="Exit Disclaimer logo" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/wikicontent/Exit%20Disclaimer%20to%20Include/exit_small.png?api=v2">
</a>
Once you&#39;ve fetched your microarray data from caArray from any of the above versions of the caArrayImporter, you can use that data in any GenePattern module or pipeline that accepts gene expression data. To view compatible modules from within GenePattern you can click the context arrow next to the imported file (the result file of your caArrayImporter job in Recent Jobs) to see which modules can accept this dataset.Help Downloading FilesFor help accessing PDF, audio, video, and compressed files on this wiki, go to
<a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=328991650" rel="nofollow">
  Help Downloading Files
</a>
.