<h1 id="caArray095ListsofMustHaveandNicetoHaveFeaturestoSupportNextGenerationSequencingData-Question:WhatAreSomeProposedNext-GenerationSequencingFeaturesforFuturecaArrayReleases?">
  Question: What Are Some Proposed Next-Generation Sequencing Features for Future caArray Releases?
</h1>
Topic: caArray Usage


Release: future caArray release



Date entered: 05/03/2012
<h2 id="caArray095ListsofMustHaveandNicetoHaveFeaturestoSupportNextGenerationSequencingData-Answer">
  Answer
</h2>
Several research groups have inquired about adding support to next-generation sequencing data in caArray. We have hosted two requirements gathering meetings with potential NGS data users at Columbia University. The users we interviewed belong to two different categories and represent different types of needs. One represents the core facilities and the other represents investigator labs that are heavy users of NGS data. The following two tables summarize features that they deem important. Please add your comments and features you would like to have to the corresponding forum thread.
<h3 id="caArray095ListsofMustHaveandNicetoHaveFeaturestoSupportNextGenerationSequencingData-Must-HaveFeatures">
  Must-Have Features
</h3>
<table>
  <tbody>
    <tr>
      <td>
        Features
      </td>
      <td>
        Explanations
      </td>
    </tr>
    <tr>
      <td>
        Support FASTQ, BAM, and VCF   Formats
      </td>
      <td>
        NGS data are stored in   those three formats. A FASTQ file contains both base calls and quality scores   for sequence reads. It is usually compressed as gz file. A BAM file contains   the alignment of multiple reads against a reference genome, as well as base   calls and quality scores for each sequence read. It is usually compressed as   well. The VCF file contains variant calls.
      </td>
    </tr>
    <tr>
      <td>
        Support large files (100GB)
      </td>
      <td>
        A FASTQ file is typically   100GB in size for whole genome sequencing and 5GB in size for exon sequencing   (at medium depth coverage). A BAM file is normally around 100 GB.
      </td>
    </tr>
    <tr>
      <td>
        Need to track capture protocol   and sequencing platform
      </td>
      <td>
        For many applications   (e.g., exome sequencing, ChIP-seq) specific target genomic DNA regions (e.g.,   exons) have to be captured and sequenced. Knowledge of both the capture   protocol and the sequencing platform is important.
      </td>
    </tr>
    <tr>
      <td>
        Need to save multiple   protocols for RNA-seq
      </td>
      <td>
        There are many protocols that   can be used to capture and filter RNA, such as ribosome RNA reduction, polyA   pull-down. All applicable protocols should be saved along with an   experiment/project.
      </td>
    </tr>
    <tr>
      <td>
        Need to capture the number   of reads and the depth of coverage
      </td>
      <td>
        Metadata about whatâ€™s in a   FASTQ and/or BAM file is informative for researchers.
      </td>
    </tr>
    <tr>
      <td>
        Support both single-end   sequencing and paired-end sequencing
      </td>
      <td>
        For single-end sequencing,   one FASTQ and one BAM file are produced. For paired-end sequencing, two FASTQ   and one BAM file are produced.
      </td>
    </tr>
    <tr>
      <td>
        Support many-to-one mapping   between FASTQ files and samples.
      </td>
      <td>
        For whole genome   sequencing, usually several FASTQ files are produced for a single sample.
      </td>
    </tr>
    <tr>
      <td>
        Record the version of the reference   genome sequence used in read alignment
      </td>
      <td>
        The version of the   reference genome used for mapping and base calling is extremely important and   should be provided for each BAM file.
      </td>
    </tr>
    <tr>
      <td>
        Support for cufflinks   output file
      </td>
      <td>
        It is desirable to be able   to retrieve FPKM data for genes or transcripts.
      </td>
    </tr>
  </tbody>
</table>
<h3 id="caArray095ListsofMustHaveandNicetoHaveFeaturestoSupportNextGenerationSequencingData-Nice-To-HaveFeatures">
  Nice-To-Have Features
</h3>
<table>
  <tbody>
    <tr>
      <td>
        Features
      </td>
      <td>
        Explanations
      </td>
    </tr>
    <tr>
      <td>
        Connect to Galaxy or other   data analysis tools
      </td>
      <td>
        Once data are stored in   caArray, this connectivity can make it easy for researchers to conduct   subsequent data analysis.
      </td>
    </tr>
    <tr>
      <td>
        Ability to search with an   aggregate of attributes
      </td>
      <td>
        Right now caArray only   allows searching with one attribute such as experiment title, sample name,   etc. It will be nice to enable search with multiple attributes.
      </td>
    </tr>
    <tr>
      <td>
        Ability to download through   Aspera or something more efficient than ftp for large archives and perform   MD5 checksum
      </td>
      <td>
        Data uploading and   downloading will be time consuming given the file size. Faster is always   better.
      </td>
    </tr>
  </tbody>
</table>
<h2 id="caArray095ListsofMustHaveandNicetoHaveFeaturestoSupportNextGenerationSequencingData-Haveacomment?">
  Have a comment?
</h2>
Please leave your comment in the
<a href="https://cabig-kc.nci.nih.gov/Molecular/forums/viewtopic.php?f=6&amp;t=773&amp;start=0&amp;sid=7bcdeb3fa07b3e124d2d561ff1124731" rel="nofollow">
  caArray End User Forum
</a>
. 