<h1 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Question:HowdoIuploadMicroArrayGeneExpressionDataintocaArray?">
  Question: How do I upload MicroArray Gene Expression Data into caArray?
</h1>
Topic: caArray Usage

Release: caArray 2.0 and above

Date entered: 01/06/2009

I want to load data related to a microarray gene expression experiment into caArray. Where should each piece of data go in caArray?
<h2 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Answer">
  Answer
</h2>
A microarray gene expression experiment is typically associated with the following types of files: array design files, sample annotation files, and raw and/or processed data files. In addition, a user might also want to associate with a particular experiment, files that describe the experiment, files that describe the samples, and other miscellaneous files.

caArray allows a user to load all types of files into the system. Furthermore, caArray performs data parsing for some of those files if it has the corresponding data parsing modules installed. The
<a href="https://wiki.nci.nih.gov/x/LBo9Ag" rel="nofollow">
  caArray User&#39;s Guide
</a>
has detailed descriptions of the types of files supported by caArray and steps taken to upload, validate, and import those files. See
<a href="https://wiki.nci.nih.gov/x/Oxo9Ag#7-SubmittingDatatoanExperiment-AboutFileTypesincaArray" rel="nofollow">
  About File Types
</a>
.

Given that in most cases the user uploads data generated from one particular array platform at a time, this document provides a step-wise description of the data upload/validate/import process in caArray for each supported array platform.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step1.(Forallarrayplatforms):ChecktheavailabilityofArrayDesignFile(s)">
  Step 1. (For all array platforms): Check the availability of Array Design File(s)
</h3>
Before an experiment is created in caArray, you should first check whether the corresponding Array Design File(s) already exist in caArray. To do so, select  Manage Array Designs under Curation on the left sidebar, as shown in the image below.

Manage Array Designs Screen Listing the Design Files
<img alt="Screenshot showing Manage Array Designs Screen Listing the Design Files" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/800px-Manage-Arrary-Design.jpg?api=v2">
If the corresponding Array Design File(s) for the array platform used in the experiment are found, proceed to Step 2.

Manage Array Designs Screen with Import a New Array Design Link
<img alt="Screenshot showing Manage Array Designs Screen with Import a New Array Design Link" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/Upload-Arrary-Design.jpg?api=v2">
If they are not there, you should click Import a New Array Design as shown in the preceding image, and follow the process described in the diagram below to import an Array Design File. (Only a registered user can import an Array Design File).

Diagram of Process Flow to Import an Array Design File
<img alt="Diagram of Process Flow to Import an Array Design File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/472px-Import-Array-Design.jpg?api=v2">
As shown in the figure above, to begin importing a file an array design file into caArray, first select Manage Array Designs under Curation in the left sidebar. Then select Import a New Array Design button at the top right of the Manage Array Designs page. The following file formats can be both imported and parsed by caArray: Affymetrix CDF, PGF, and CLF; Illumina Design CSV; and Genepix Design GAL. The following file formats can be imported, but not parsed: Affymetrix CSV, XML MAGE-TAB, and ADF; ImaGene TPL; Nimblegen NDF; and UCSF SPOT SPT.

Note that any array design format not mentioned in the flow diagram is not supported by caArray up to version 2.1.1.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step2(forallarrayplatforms):Createanexperiment">
  Step 2 (for all array platforms): Create an experiment
</h3>
<a href="https://wiki.nci.nih.gov/x/LBo9Ag" rel="nofollow">
  The caArray User&#39;s Guide
</a>
provides detailed information for
<a href="https://wiki.nci.nih.gov/x/Nho9Ag#5-CreatingandManagingExperiments-CreatinganExperiment" rel="nofollow">
  creating an experiment
</a>
.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3:Upload,validate,andimportfilesintocaArray">
  Step 3: Upload, validate, and import files into caArray
</h3>
In this step, slightly different processes are used to load data into caArray for different array platforms.
<ul>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3A Import data under the MAGE-TAB format
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3B Import experimental data using the Affymetrix platform
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3C Import experimental data using the Illumina platform
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3D Import experimental data using the Genepix platform
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3E Import experimental data using the Agilent platform
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3F Import experimental data using the ImaGene platform
    </a>
  </li>
  <li>
    <a href="https://wiki.nci.nih.gov/pages/viewpage.action?pageId=42832243" rel="nofollow">
      #Step 3G Import experimental data using the Nimblegen platform
    </a>
  </li>
</ul>
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3AImportdataundertheMAGE-TABformat">
  Step 3A Import data under the MAGE-TAB format
</h3>
Only file types listed in the flow diagram below can be imported.

Diagram of Process Flow to Import a MAGE-TAB File
<img alt="Diagram of Process Flow to Import a MAGE-TAB File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/325px-Import-MAGE-data.JPG?api=v2">
According to the figure above, to import a MAGE-TAB file, first select the Data tab on the Experiment Details page, then proceed to upload and validate the file you wish to import. The following MAGE-TAB file formats can be both imported and parsed by caArray: SDRF and IDF, along with their associated data files (Affymetrix CEL and CHP). MAGE-TAB data matrix file formats can be imported, but not parsed.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3BImportexperimentaldatausingtheAffymetrixplatform">
  Step 3B Import experimental data using the Affymetrix platform
</h3>
Because of a known issue concerning importing Affymetrix DAT files, uploaded Affymetrix DAT files should be kept in the &#34;Manage Data&#34; tab under the &#34;Data&#34; tab for a selected experiment for caArray versions up to version 2.1.1. Once the issue is resolved, Affymetrix DAT files will be able to be imported the same way as EXP, RPT, and TXT files.

Diagram of Process Flow to Import an Affymetrix DAT File
<img alt="Diagram of Process Flow to Import an Affymetrix DAT File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/443px-Import-Affymetrix-data.jpg?api=v2">
According to the figure above, to import files containing experimental data from the Affymetrix platform, first select the Data tab on the Experiment Details page, then proceed to upload the file. For a CEL or CHP file, you should validate the file, then import and implicitly parse it. For an EXP, RPT, or TXT file, you should validate the file, then import it. DAT files should not be validated or imported; instead they should be kept under the Manage Data tab. You should assign all other file formats to the ‘Supplementary Files’ category.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3CImportexperimentaldatausingtheIlluminaplatform">
  Step 3C Import experimental data using the Illumina platform
</h3>
Diagram of Process Flow to Import an Illumina Data File
<img alt="Diagram of Process Flow to Import an Illumina Data File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/424px-Import-Illumina-data.JPG?api=v2">
According to the figure above, to import files containing experimental data from the Illumina platform, first select the Data tab on the ‘Experiment Details’ page, then proceed to upload the file. For a Data CVS file, you should validate the file, then import and implicitly parse it. For an IDAT or TXT file, you should validate the file, then import it. You should assign all other file formats to the Supplementary Files category.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3DImportexperimentaldatausingtheGenepixplatform">
  Step 3D Import experimental data using the Genepix platform
</h3>
Diagram of Process Flow to Import a Genepix Data File
<img alt="Diagram of Process Flow to Import a Genepix Data File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/263px-Import-Genepix-data.jpg?api=v2">
According to the figure above, to import files containing experimental data from the Genepix platform, first select the Data tab on the Experiment Details page, then proceed to upload the file. For a GPR file, you should validate the file, then import and implicitly parse it. You should assign all other file formats to the Supplementary Files category.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3EImportexperimentaldatausingtheAgilentplatform">
  Step 3E Import experimental data using the Agilent platform
</h3>
Because of a known issue concerning importing Agilent TSV files, uploaded Agilent TSV files should remain in the &#34;Manage Data&#34; tab under the &#34;Data&#34; tab for a selected experiment for caArray versions up to version 2.1.1. Once the issue is resolved, Agilent TSV files will be able to be imported the same way as TXT files.

Diagram of Process Flow to Import an Agilent Data File
<img alt="Diagram of Process Flow to Import an Agilent Data File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/442px-Import-Agilent-data.jpg?api=v2">
According to the figure above, to import files containing experimental data from the Agilent platform, first select the Data tab on the Experiment Details page, then proceed to upload the file. For a TXT file, you should validate the file, then import it. TSV files should not be validated or imported. They should instead be kept under the Manage Data tab. You should assign all other file formats to the Supplementary Files category.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3FImportexperimentaldatausingtheImaGeneplatform">
  Step 3F Import experimental data using the ImaGene platform
</h3>
Because of a known issue concerning importing ImaGene TIF files, uploaded ImaGene TIF files should remain in the &#34;Manage Data&#34; tab under the &#34;Data&#34; tab for a selected experiment for caArray versions up to version 2.1.1. Once the issue is resolved, ImaGene TIF files will be able to be imported the same way as TXT files.

Diagram of Process Flow to Import an ImaGene Data File
<img alt="Diagram of Process Flow to Import an ImaGene Data File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/451px-Import-ImaGene-data.jpg?api=v2">
According to the figure above, to import files containing experimental data from the ImaGene platform, first select the Data tab on the Experiment Details page, then proceed to upload the file. For a TXT file, you should validate the file, then import it. TIF files should not be validated or imported, and should instead be kept under the Manage Data tab. You should assign all other file formats to the Supplementary Files category.
<h3 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Step3GImportexperimentaldatausingtheNimblegenplatform">
  Step 3G Import experimental data using the Nimblegen platform
</h3>
Diagram of Process Flow to Import a Nimblegen Data File
<img alt="Diagram of Process Flow to Import a Nimblegen Data File" src="https://wiki.nci.nih.gov/download/attachments/embedded-page/caArray2/caArray%20002%20-%20Uploading%20MicroArray%20Gene%20Expression%20Data%20into%20caArray/336px-Import-Nimblegen-data.jpg?api=v2">
According to the figure above, to import files containing experimental data from the Nimblegen platform, first select the Data tab on the Experiment Details page, then proceed to upload the file. For a TXT or GFF file, you should validate the file, then import it. You should assign all other file formats to the Supplementary Files category.
<h2 id="caArray002UploadingMicroArrayGeneExpressionDataintocaArray-Haveacomment?">
  Have a comment?
</h2>
Please leave your comment in the
<a href="https://cabig-kc.nci.nih.gov/Molecular/forums/viewtopic.php?f=6&amp;t=577" rel="nofollow">
  caArray End User Forum
</a>
. 